const global_cache_data = [["DS_q4h7q8q0fnve_0",{"n":"23-Single-Element-DNPs_RSCDD_2023-Ag","d":"Configurations of Ag from Andolina & Saidi, 2023. One of 23 minimalist, curated sets of DFT-calculated properties for individual elements for the purpose of providing input to machine learning of deep neural network potentials (DNPs). Each element set contains on average ~4000 structures with 27 atoms per structure. Configuration metadata includes Materials Project ID where available, as well as temperatures at which MD trajectories were calculated.These temperatures correspond to the melting temperature (MT) and 0.25*MT for elements with MT < 2000K, and MT, 0.6*MT and 0.25*MT for elements with MT > 2000K."}],["DS_8y775we7um7w_0",{"n":"23-Single-Element-DNPs_RSCDD_2023-Al","d":"Configurations of Al from Andolina & Saidi, 2023. One of 23 minimalist, curated sets of DFT-calculated properties for individual elements for the purpose of providing input to machine learning of deep neural network potentials (DNPs). Each element set contains on average ~4000 structures with 27 atoms per structure. Configuration metadata includes Materials Project ID where available, as well as temperatures at which MD trajectories were calculated.These temperatures correspond to the melting temperature (MT) and 0.25*MT for elements with MT < 2000K, and MT, 0.6*MT and 0.25*MT for elements with MT > 2000K."}],["DS_iie3c31ar46x_0",{"n":"23-Single-Element-DNPs_RSCDD_2023-Au","d":"Configurations of Au from Andolina & Saidi, 2023. One of 23 minimalist, curated sets of DFT-calculated properties for individual elements for the purpose of providing input to machine learning of deep neural network potentials (DNPs). Each element set contains on average ~4000 structures with 27 atoms per structure. Configuration metadata includes Materials Project ID where available, as well as temperatures at which MD trajectories were calculated.These temperatures correspond to the melting temperature (MT) and 0.25*MT for elements with MT < 2000K, and MT, 0.6*MT and 0.25*MT for elements with MT > 2000K."}],["DS_jt0lax9yd15r_0",{"n":"23-Single-Element-DNPs_RSCDD_2023-Co","d":"Configurations of Co from Andolina & Saidi, 2023. One of 23 minimalist, curated sets of DFT-calculated properties for individual elements for the purpose of providing input to machine learning of deep neural network potentials (DNPs). Each element set contains on average ~4000 structures with 27 atoms per structure. Configuration metadata includes Materials Project ID where available, as well as temperatures at which MD trajectories were calculated.These temperatures correspond to the melting temperature (MT) and 0.25*MT for elements with MT < 2000K, and MT, 0.6*MT and 0.25*MT for elements with MT > 2000K."}],["DS_dc3o40aou2le_0",{"n":"23-Single-Element-DNPs_RSCDD_2023-Cu","d":"Configurations of Cu from Andolina & Saidi, 2023. One of 23 minimalist, curated sets of DFT-calculated properties for individual elements for the purpose of providing input to machine learning of deep neural network potentials (DNPs). Each element set contains on average ~4000 structures with 27 atoms per structure. Configuration metadata includes Materials Project ID where available, as well as temperatures at which MD trajectories were calculated.These temperatures correspond to the melting temperature (MT) and 0.25*MT for elements with MT < 2000K, and MT, 0.6*MT and 0.25*MT for elements with MT > 2000K."}],["DS_90fnsmavv1am_0",{"n":"23-Single-Element-DNPs_RSCDD_2023-Ge","d":"Configurations of Ge from Andolina & Saidi, 2023. One of 23 minimalist, curated sets of DFT-calculated properties for individual elements for the purpose of providing input to machine learning of deep neural network potentials (DNPs). Each element set contains on average ~4000 structures with 27 atoms per structure. Configuration metadata includes Materials Project ID where available, as well as temperatures at which MD trajectories were calculated.These temperatures correspond to the melting temperature (MT) and 0.25*MT for elements with MT < 2000K, and MT, 0.6*MT and 0.25*MT for elements with MT > 2000K."}],["DS_gc1y80tpyylb_0",{"n":"23-Single-Element-DNPs_RSCDD_2023-I","d":"Configurations of I from Andolina & Saidi, 2023. One of 23 minimalist, curated sets of DFT-calculated properties for individual elements for the purpose of providing input to machine learning of deep neural network potentials (DNPs). Each element set contains on average ~4000 structures with 27 atoms per structure. Configuration metadata includes Materials Project ID where available, as well as temperatures at which MD trajectories were calculated.These temperatures correspond to the melting temperature (MT) and 0.25*MT for elements with MT < 2000K, and MT, 0.6*MT and 0.25*MT for elements with MT > 2000K."}],["DS_omnl1yy49sdh_0",{"n":"23-Single-Element-DNPs_RSCDD_2023-Kr","d":"Configurations of Kr from Andolina & Saidi, 2023. One of 23 minimalist, curated sets of DFT-calculated properties for individual elements for the purpose of providing input to machine learning of deep neural network potentials (DNPs). Each element set contains on average ~4000 structures with 27 atoms per structure. Configuration metadata includes Materials Project ID where available, as well as temperatures at which MD trajectories were calculated.These temperatures correspond to the melting temperature (MT) and 0.25*MT for elements with MT < 2000K, and MT, 0.6*MT and 0.25*MT for elements with MT > 2000K."}],["DS_wyo91w20wlgm_0",{"n":"23-Single-Element-DNPs_RSCDD_2023-Li","d":"Configurations of Li from Andolina & Saidi, 2023. One of 23 minimalist, curated sets of DFT-calculated properties for individual elements for the purpose of providing input to machine learning of deep neural network potentials (DNPs). Each element set contains on average ~4000 structures with 27 atoms per structure. Configuration metadata includes Materials Project ID where available, as well as temperatures at which MD trajectories were calculated.These temperatures correspond to the melting temperature (MT) and 0.25*MT for elements with MT < 2000K, and MT, 0.6*MT and 0.25*MT for elements with MT > 2000K."}],["DS_mevqyitwxukc_0",{"n":"23-Single-Element-DNPs_RSCDD_2023-Mg","d":"Configurations of Mg from Andolina & Saidi, 2023. One of 23 minimalist, curated sets of DFT-calculated properties for individual elements for the purpose of providing input to machine learning of deep neural network potentials (DNPs). Each element set contains on average ~4000 structures with 27 atoms per structure. Configuration metadata includes Materials Project ID where available, as well as temperatures at which MD trajectories were calculated.These temperatures correspond to the melting temperature (MT) and 0.25*MT for elements with MT < 2000K, and MT, 0.6*MT and 0.25*MT for elements with MT > 2000K."}],["DS_bjkftn0ug9r3_0",{"n":"23-Single-Element-DNPs_RSCDD_2023-Mo","d":"Configurations of Mo from Andolina & Saidi, 2023. One of 23 minimalist, curated sets of DFT-calculated properties for individual elements for the purpose of providing input to machine learning of deep neural network potentials (DNPs). Each element set contains on average ~4000 structures with 27 atoms per structure. Configuration metadata includes Materials Project ID where available, as well as temperatures at which MD trajectories were calculated.These temperatures correspond to the melting temperature (MT) and 0.25*MT for elements with MT < 2000K, and MT, 0.6*MT and 0.25*MT for elements with MT > 2000K."}],["DS_zbxayq0diq6l_0",{"n":"23-Single-Element-DNPs_RSCDD_2023-Nb","d":"Configurations of Nb from Andolina & Saidi, 2023. One of 23 minimalist, curated sets of DFT-calculated properties for individual elements for the purpose of providing input to machine learning of deep neural network potentials (DNPs). Each element set contains on average ~4000 structures with 27 atoms per structure. Configuration metadata includes Materials Project ID where available, as well as temperatures at which MD trajectories were calculated.These temperatures correspond to the melting temperature (MT) and 0.25*MT for elements with MT < 2000K, and MT, 0.6*MT and 0.25*MT for elements with MT > 2000K."}],["DS_nue14kckbkdh_0",{"n":"23-Single-Element-DNPs_RSCDD_2023-Ni","d":"Configurations of Ni from Andolina & Saidi, 2023. One of 23 minimalist, curated sets of DFT-calculated properties for individual elements for the purpose of providing input to machine learning of deep neural network potentials (DNPs). Each element set contains on average ~4000 structures with 27 atoms per structure. Configuration metadata includes Materials Project ID where available, as well as temperatures at which MD trajectories were calculated.These temperatures correspond to the melting temperature (MT) and 0.25*MT for elements with MT < 2000K, and MT, 0.6*MT and 0.25*MT for elements with MT > 2000K."}],["DS_098x6q7kbeat_0",{"n":"23-Single-Element-DNPs_RSCDD_2023-Os","d":"Configurations of Os from Andolina & Saidi, 2023. One of 23 minimalist, curated sets of DFT-calculated properties for individual elements for the purpose of providing input to machine learning of deep neural network potentials (DNPs). Each element set contains on average ~4000 structures with 27 atoms per structure. Configuration metadata includes Materials Project ID where available, as well as temperatures at which MD trajectories were calculated.These temperatures correspond to the melting temperature (MT) and 0.25*MT for elements with MT < 2000K, and MT, 0.6*MT and 0.25*MT for elements with MT > 2000K."}],["DS_k065jfggbq43_0",{"n":"23-Single-Element-DNPs_RSCDD_2023-Pb","d":"Configurations of Pb from Andolina & Saidi, 2023. One of 23 minimalist, curated sets of DFT-calculated properties for individual elements for the purpose of providing input to machine learning of deep neural network potentials (DNPs). Each element set contains on average ~4000 structures with 27 atoms per structure. Configuration metadata includes Materials Project ID where available, as well as temperatures at which MD trajectories were calculated.These temperatures correspond to the melting temperature (MT) and 0.25*MT for elements with MT < 2000K, and MT, 0.6*MT and 0.25*MT for elements with MT > 2000K."}],["DS_g0sb0h7usqw7_0",{"n":"23-Single-Element-DNPs_RSCDD_2023-Pd","d":"Configurations of Pd from Andolina & Saidi, 2023. One of 23 minimalist, curated sets of DFT-calculated properties for individual elements for the purpose of providing input to machine learning of deep neural network potentials (DNPs). Each element set contains on average ~4000 structures with 27 atoms per structure. Configuration metadata includes Materials Project ID where available, as well as temperatures at which MD trajectories were calculated.These temperatures correspond to the melting temperature (MT) and 0.25*MT for elements with MT < 2000K, and MT, 0.6*MT and 0.25*MT for elements with MT > 2000K."}],["DS_0zgz34a90a6i_0",{"n":"23-Single-Element-DNPs_RSCDD_2023-Pt","d":"Configurations of Pt from Andolina & Saidi, 2023. One of 23 minimalist, curated sets of DFT-calculated properties for individual elements for the purpose of providing input to machine learning of deep neural network potentials (DNPs). Each element set contains on average ~4000 structures with 27 atoms per structure. Configuration metadata includes Materials Project ID where available, as well as temperatures at which MD trajectories were calculated.These temperatures correspond to the melting temperature (MT) and 0.25*MT for elements with MT < 2000K, and MT, 0.6*MT and 0.25*MT for elements with MT > 2000K."}],["DS_gzbvdicu231p_0",{"n":"23-Single-Element-DNPs_RSCDD_2023-Re","d":"Configurations of Re from Andolina & Saidi, 2023. One of 23 minimalist, curated sets of DFT-calculated properties for individual elements for the purpose of providing input to machine learning of deep neural network potentials (DNPs). Each element set contains on average ~4000 structures with 27 atoms per structure. Configuration metadata includes Materials Project ID where available, as well as temperatures at which MD trajectories were calculated.These temperatures correspond to the melting temperature (MT) and 0.25*MT for elements with MT < 2000K, and MT, 0.6*MT and 0.25*MT for elements with MT > 2000K."}],["DS_z90lfjg88fzo_0",{"n":"23-Single-Element-DNPs_RSCDD_2023-Sb","d":"Configurations of Sb from Andolina & Saidi, 2023. One of 23 minimalist, curated sets of DFT-calculated properties for individual elements for the purpose of providing input to machine learning of deep neural network potentials (DNPs). Each element set contains on average ~4000 structures with 27 atoms per structure. Configuration metadata includes Materials Project ID where available, as well as temperatures at which MD trajectories were calculated.These temperatures correspond to the melting temperature (MT) and 0.25*MT for elements with MT < 2000K, and MT, 0.6*MT and 0.25*MT for elements with MT > 2000K."}],["DS_o3itca7mk80r_0",{"n":"23-Single-Element-DNPs_RSCDD_2023-Sr","d":"Configurations of Sr from Andolina & Saidi, 2023. One of 23 minimalist, curated sets of DFT-calculated properties for individual elements for the purpose of providing input to machine learning of deep neural network potentials (DNPs). Each element set contains on average ~4000 structures with 27 atoms per structure. Configuration metadata includes Materials Project ID where available, as well as temperatures at which MD trajectories were calculated.These temperatures correspond to the melting temperature (MT) and 0.25*MT for elements with MT < 2000K, and MT, 0.6*MT and 0.25*MT for elements with MT > 2000K."}],["DS_ofpcyxez6xsc_0",{"n":"23-Single-Element-DNPs_RSCDD_2023-Ti","d":"Configurations of Ti from Andolina & Saidi, 2023. One of 23 minimalist, curated sets of DFT-calculated properties for individual elements for the purpose of providing input to machine learning of deep neural network potentials (DNPs). Each element set contains on average ~4000 structures with 27 atoms per structure. Configuration metadata includes Materials Project ID where available, as well as temperatures at which MD trajectories were calculated.These temperatures correspond to the melting temperature (MT) and 0.25*MT for elements with MT < 2000K, and MT, 0.6*MT and 0.25*MT for elements with MT > 2000K."}],["DS_xsad0btc0bsn_0",{"n":"23-Single-Element-DNPs_RSCDD_2023-Zn","d":"Configurations of Zn from Andolina & Saidi, 2023. One of 23 minimalist, curated sets of DFT-calculated properties for individual elements for the purpose of providing input to machine learning of deep neural network potentials (DNPs). Each element set contains on average ~4000 structures with 27 atoms per structure. Configuration metadata includes Materials Project ID where available, as well as temperatures at which MD trajectories were calculated.These temperatures correspond to the melting temperature (MT) and 0.25*MT for elements with MT < 2000K, and MT, 0.6*MT and 0.25*MT for elements with MT > 2000K."}],["DS_h9w8v7289utq_0",{"n":"23-Single-Element-DNPs_RSCDD_2023-Zr","d":"Configurations of Zr from Andolina & Saidi, 2023. One of 23 minimalist, curated sets of DFT-calculated properties for individual elements for the purpose of providing input to machine learning of deep neural network potentials (DNPs). Each element set contains on average ~4000 structures with 27 atoms per structure. Configuration metadata includes Materials Project ID where available, as well as temperatures at which MD trajectories were calculated.These temperatures correspond to the melting temperature (MT) and 0.25*MT for elements with MT < 2000K, and MT, 0.6*MT and 0.25*MT for elements with MT > 2000K."}],["DS_gaq0fec8i6ik_0",{"n":"3BPA_isolated_atoms","d":"Reference C, H, O, and N atoms from 3BPA, used to showcase the performance of linear atomic cluster expansion (ACE) force fields in a machine learning model to predict the potential energy surfaces of organic molecules."}],["DS_xaasqmrdv28s_0",{"n":"3BPA_test_1200K","d":"Test configurations with MD simulations performed at 1200K from 3BPA, used to showcase the performance of linear atomic cluster expansion (ACE) force fields in a machine learning model to predict the potential energy surfaces of organic molecules."}],["DS_i4hwummq4c7j_0",{"n":"3BPA_test_300K","d":"Test configurations with MD simulations performed at 300K from 3BPA, used to showcase the performance of linear atomic cluster expansion (ACE) force fields in a machine learning model to predict the potential energy surfaces of organic molecules."}],["DS_epqxcavda3kt_0",{"n":"3BPA_test_600K","d":"Test configurations with MD simulations performed at 600K from 3BPA, used to showcase the performance of linear atomic cluster expansion (ACE) force fields in a machine learning model to predict the potential energy surfaces of organic molecules."}],["DS_8cg3pdvxt0pa_0",{"n":"3BPA_test_dih_beta120","d":"Test configurations with fixed value for dihedral beta in alpha-gamma  plane of 120 degreesfrom 3BPA dataset. Used to showcase the performance of linear atomic cluster expansion (ACE) force fields in a machine learning model to predict the potential energy surfaces of organic molecules."}],["DS_199ama4h9t7m_0",{"n":"3BPA_test_dih_beta150","d":"Test configurations with fixed value for dihedral beta in alpha-gamma  plane of 150 degreesfrom 3BPA dataset. Used to showcase the performance of linear atomic cluster expansion (ACE) force fields in a machine learning model to predict the potential energy surfaces of organic molecules."}],["DS_ilijykl0t0fn_0",{"n":"3BPA_test_dih_beta180","d":"Test configurations with fixed value for dihedral beta in alpha-gamma  plane of 180 degreesfrom 3BPA dataset. Used to showcase the performance of linear atomic cluster expansion (ACE) force fields in a machine learning model to predict the potential energy surfaces of organic molecules."}],["DS_hu0btdblv8x6_0",{"n":"3BPA_train_300K","d":"Training configurations with MD simulations performed at 300K from 3BPA, used to showcase the performance of linear atomic cluster expansion (ACE) force fields in a machine learning model to predict the potential energy surfaces of organic molecules."}],["DS_v88913tvdxbr_0",{"n":"3BPA_train_mixed","d":"Training configurations with MD simulation performed at 300K, 600K and 1200K from 3BPA dataset, used to showcase the performance of linear atomic cluster expansion (ACE) force fields in a machine learning model to predict the potential energy surfaces of organic molecules."}],["DS_0ady7a8a8n6p_0",{"n":"ABC2D6-16_PRL_2018","d":"Dataset used to train a machine learning model to calculate density functional theory-quality formation energies of all ~2 x 106 pristine ABC2D6 elpasolite crystals that can be made up from main-group elements (up to bismuth)."}],["DS_0nnbymlcjota_0",{"n":"AENET_amorphous_LiSi_JCP2021","d":"The amorphous LiSi data set comprises 45,169 atomic structures with compositions Li(x)Si (0.0\u2264x\u22644.75) and the corresponding energies and interatomic forces, which were generated using an iterative approach based on an evolutionary algorithm and subsequent refinement, as described in detail in reference [15]. The data includes bulk, surface, and cluster structures with system sizes of up to 608 atoms. The energies and forces of the LiSi structures were obtained from DFT calculations using the Perdew-Burke-Ernzerhof [10] exchange-correlation functional and projector-augmented wave pseudopotentials [16], as implemented in the Vienna Ab-Initio Simulation Package (VASP) [17,18]. We employed a plane-wave basis set with an energy cutoff of 520 eV for the representation of the wavefunctions and a uniform gamma-centered k-point grid for the Brillouin zone integration, with a mesh density corresponding to a number of k points of at least 1000 divided by the number of atoms. The atomic positions and lattice parameters of all structures were optimized until residual forces were below 20 meV/\u00c5. This dataset was also used for the construction of the ANN potential in Ref. [15] and [19]. [10] J. P. Perdew, K. Burke, and M. Ernzerhof, Phys. Rev. Lett. 77, 3865 (1996). [15] N. Artrith, A. Urban, G. Ceder, J. Chem. Phys. 148 (2018) 241711. [16] P. E. Bl\u00f6chl, Phys. Rev. B 50, 17953\u201317979 (1994). [17] G. Kresse, J. Furthm\u00fcller, Phys. Rev. B 54, 11169\u201311186 (1996). [18] Kresse, J. Furthm\u00fcller, Comput. Mater. Sci. 6, 15\u201350 (1996). [19] N. Artrith, A. Urban, Y. Wang, G. Ceder, arXiv:1901.09272, https://arxiv.org/pdf/1901.09272.pdf"}],["DS_v4bk24pq0had_0",{"n":"AENET_liquid_water_dataset_JCP2021","d":"The water data set comprises energies and forces of 9,189 condensed-phase structures. The data was obtained in an iterative procedure described in detail in Ref. [4]. The final ANN potential was employed in Refs. [4,5] to analyze temperature-dependent Raman spectra of liquid water. The data set contains structures from four iterations: Initial structures (iteration 0) were obtained from classical and path integral AIMD simulations of bulk liquid water in a cubic box containing 64 water molecules at 300 K as reported in Ref. [6]. Distorted configurations with higher forces were added by randomly displacing the Cartesian coordinates of these configurations. Iteration 1 contains a set of 500 configurations from MD simulations with the fully flexible SPC/E flex water model [7] employing a 25 % increased water density (simulation box with 80 water molecules) and elevated temperatures (T = 500 K) in order to sample highly repulsive configurations. Structures in iteration 2 were obtained by classical MD simulations with preliminary ANN potentials at T = 300 K, 325 K, 350 K, and 370 K employing cubic boxes with 64 molecules and the corresponding experimental densities. The final iteration 3 data contains structures from preliminary ANN simulations with classical and quantum nuclei, respectively, at a wide range of temperatures (T = 258 K, 268 K, 280 K, 290 K, 300 K, 310 K, 320 K, 330 K, 340 K, 350 K, 360 K, and 370 K) using cubic boxes with 64 molecules and the corresponding experimental densities. Energies and atomic forces were calculated with the CP2K program [8,9] using the revPBE exchange-correlation functional [10,11] with D3 dispersion correction [12] following the setup reported in Ref. [4]. Atomic cores were represented using the dual-space Goedecker-Teter-Hutter pseudopotentials [13], Kohn-Sham orbitals were expanded in the TZV2P basis set within the GPW method [14], and the density was represented by an auxiliary plane-wave basis with a cutoff of 400 Ry. [1] A. Kokalj, J. Mol. Graphics Modell. 17, 176\u2013179 (1999). [2] N. Artrith, A. Urban, Comput. Mater. Sci. 114, 135\u2013150 (2016). [3] N. Artrith, A. Urban, G. Ceder, Phys. Rev. B 96, 014112 (2017). [4] T. Morawietz, O. Marsalek, S. R. Pattenaude, L. M. Streacker, D. Ben-Amotz, and T. E. Markland, J. Phys. Chem. Lett. 9, 851 (2018). [5] T. Morawietz, A. S. Urbina, P. K. Wise, X. Wu, W. Lu, D. Ben-Amotz, and T. E. Markland, J. Phys. Chem. Lett. 10, 6067 (2019). [6] Marsalek and T. E. Markland, J. Phys. Chem. Lett. 8, 1545 (2017). [7] X. B. Zhang, Q. L. Liu, and A. M. Zhu, Fluid Ph. Equilibria 262, 210(2007). [8] J. VandeVondele, M. Krack, F. Mohamed, M. Parrinello, T. Chassaing, and J. Hutter, Comput. Phys. Commun. 167, 103 (2005). [9] J. Hutter, M. Iannuzzi, F. Schiffmann, and J. VandeVondele, WIRES Comput. Mol. Sci. 4, 15 (2014). [10] J. P. Perdew, K. Burke, and M. Ernzerhof, Phys. Rev. Lett. 77, 3865 (1996). [11] Y. Zhang and W. Yang, Phys. Rev. Lett. 80, 890 (1998). [12] S. Grimme, J. Antony, S. Ehrlich, and H. Krieg, J. Chem. Phys. 132, 154104 (2010). [13] S. Goedecker, M. Teter, and J. Hutter, Phys. Rev. B 54, 1703 (1996). [14] B. G. Lippert, J. Hutter, and M. Parrinello, Mol. Phys. 92, 477 (1997). "}],["DS_5lhmgnxhuia3_0",{"n":"AFF_JCP_2022","d":"Approximately 145,000 configurations of alkane, aspirin, alpha-glucose and uracil, partly taken from the MD-17 dataset, used in training an 'Atomic Neural Net' model."}],["DS_p4evspy1ntcs_0",{"n":"ANI-1","d":"ANI-1 is a dataset of 20 million conformations with calculated non-equilibrium energy values. The conformations are based on a subset of the GDB-11 dataset, each molecule containing between 1 and 8 heavy atoms, with atomic species limited to C, N and O. Configuration sets are included for standard and high energy (defined as energies greater than 275 kcal*mol-1 higher than the lowest energy conformer) conformations, and, within these, number of heavy atoms per molecule."}],["DS_ko3rpzre7bea_0",{"n":"ANI-1x","d":"ANI-1x contains DFT calculations for approximately 5 million molecular conformations. From an initial training set, an active learning method was used to iteratively add conformations where insufficient diversity was detected. Additional conformations were sampled from existing databases of molecules, such as GDB-11 and ChEMBL. On each of these configurations, one of molecular dynamics sampling, normal mode sampling, dimer sampling, or torsion sampling was performed."}],["DS_gjr8gi1tb8wh_0",{"n":"ANI-2x-B973c-def2mTZVP","d":"ANI-2x-B973c-def2mTZVP is a portion of the ANI-2x dataset, which includes DFT-calculated energies for structures from 2 to 63 atoms in size containing H, C, N, O, S, F, and Cl. This portion of ANI-2x was calculated in ORCA at the B973c level of theory using the def2m-TZVP basis set. Configuration sets are divided by number of atoms per structure. Force corrections and dipoles are recorded in the metadata."}],["DS_yxeu6us2i6dh_0",{"n":"ANI-2x-wB97MD3BJ-def2TZVPP","d":"ANI-2x-wB97MD3BJ-def2TZVPP is a portion of the ANI-2x dataset, which includes DFT-calculated energies for structures from 2 to 63 atoms in size containing H, C, N, O, S, F, and Cl. This portion of ANI-2x was calculated in ORCA at the wB97M level of theory with D3 and BJ energy corrections, using the def2-TZVPP basis set. Configuration sets are divided by number of atoms per structure. Uncorrected SCF energy values and dipoles are recorded in the metadata."}],["DS_wxf8f3t3abul_0",{"n":"ANI-2x-wB97MV-def2TZVPP","d":"ANI-2x-wB97MV-def2TZVPP is a portion of the ANI-2x dataset, which includes DFT-calculated energies for structures from 2 to 63 atoms in size containing H, C, N, O, S, F, and Cl. This portion of ANI-2x was calculated at the WB97MV level of theory using the def2TZVPP basis set. Configuration sets are divided by number of atoms per structure."}],["DS_rde4chkdmm18_0",{"n":"ANI-2x-wB97X-631Gd","d":"ANI-2x-wB97X-631Gd is a portion of the ANI-2x dataset, which includes DFT-calculated energies for structures from 2 to 63 atoms in size containing H, C, N, O, S, F, and Cl. This portion of ANI-2x was calculated in Gaussian 09 at the wB97X level of theory using the 6-31G(d) basis set. Configuration sets are divided by number of atoms per structure."}],["DS_bnnyjpaqb09i_0",{"n":"ANI-Al_NC2021-test","d":"Approximately 2800 configurations from a test dataset\u2013one of a pair of train/test datasets of aluminum in crystal and melt phases, used for training and testing an ANI neural network model."}],["DS_tz8gsj3xi2g4_0",{"n":"ANI-Al_NC2021-train","d":"Approximately 2800 configurations from a train dataset\u2013one of a pair of train/test datasets of aluminum in crystal and melt phases, used for training and testing an ANI neural network model."}],["DS_6e5ljaqa3cfr_0",{"n":"Ag-PBE_MSMSE_2021","d":"Approximately 7,600 configurations of Ag used as part of a training dataset for a DP-GEN-based ML model for a Ag-Au nanoalloy potential."}],["DS_t0ec1irmeo59_0",{"n":"AgAu-nanoalloy_MSMSE_2021","d":"Approximately 50,000 configurations of Au, Ag and AuAg used as part of a training dataset for a DP-GEN-based ML model for a Ag-Au nanoalloy potential."}],["DS_y5m3hunroa7x_0",{"n":"AgPd_NPJ_2021","d":"The dataset consists of energies, forces and virials for DFT-VASP-generated Ag-Pd systems. The data was used to fit an active learned dataset which was used to compare MTP- and SOAP-GAP-generated potentials."}],["DS_vmnudsz7kx0a_0",{"n":"AlNiCu_AIP_2020","d":"This dataset is formed from two parts: single-species datasets for Al, Ni, and Cu from the NOMAD Encyclopedia and multi-species datasets that include Al, Ni and Cu from NOMAD Archive. Duplicates have been removed from NOMAD Encyclopedia data. For the multi-species data, only the last configuration steps for each NOMAD Archive record were used because the last configuration typically cooresponds with a fully relaxed configuration. In this dataset, the NOMAD unique reference access IDs are retained along with a subset of their meta information that includes whether the supplied configuration is from a converged calculation as well as the Density Functional Theory (DFT) code, version, and type of DFT functionals with the total potential energies. This dataset consists of 39.1% Al, 30.7% Ni, and 30.2% Cu and has 27,987 atomic environments in 3337 structures."}],["DS_dtjyh96dypuu_0",{"n":"AlNiTi_CMS_2019","d":"This dataset was generated using the following active learning scheme: 1) candidate structures were relaxed by a partially-trained MTP model, 2) structures for which the MTP had to perform extrapolation were passed to DFT to be re-computed, 3) the MTP was retrained, including the structures that were re-computed with DFT, 4) steps 1-3 were repeated until the MTP no longer extrapolated on any of the original candidate structures. The original candidate structures for this dataset included about 375,000 binary and ternary structures, enumerating all possible unit cells with different symmetries (BCC, FCC, and HCP) and different number of atoms."}],["DS_styimm9cal6v_0",{"n":"Al_Cu_Mg_GSFE_JMPS2019","d":"Dataset from \"Stress-dependence of generalized stacking fault energies\":DFT calculations of generalized stacking fault energies (GSFE) for Al, Cu, and Mg."}],["DS_tydu7u0h0hhz_0",{"n":"Au-PBE_MSMSE_2021","d":"Approximately 20,000 configurations of Au used as part of a  training dataset for a DP-GEN-based ML model for a Ag-Au nanoalloy  potential."}],["DS_lifzo8zpa76m_0",{"n":"BA10-18","d":"Dataset (DFT-10B) contains structures of the 10 binary alloys AgCu, AlFe, AlMg, AlNi, AlTi, CoNi, CuFe, CuNi, FeV, and NbNi. Each alloy system includes all possible unit cells with 1-8 atoms for face-centered cubic (fcc) and body-centered cubic (bcc) crystal types, and all possible unit cells with 2-8 atoms for the hexagonal close-packed (hcp) crystal type. This results in 631 fcc, 631 bcc, and 333 hcp structures, yielding 1595 x 10 = 15,950 unrelaxed structures in total. Lattice parameters for each crystal structure were set according to Vegard's law. Total energies were computed using DFT with projector-augmented wave (PAW) potentials within the generalized gradient approximation (GGA) of Perdew, Burke, and Ernzerhof (PBE) as implemented in the Vienna Ab Initio Simulation Package (VASP). The k-point meshes for sampling the Brillouin zone were constructed using generalized regular grids."}],["DS_0xfqsnoawhoo_0",{"n":"BOTnet_ACAC_2022_Dihedral_scan","d":"Dihedral scan about one of the C-C bonds of the conjugated system. Acetylacetone dataset generated from a long molecular dynamics simulation at 300 K using a Langevin thermostat at the semi-empirical GFN2-xTB level of theory. Configurations were sampled at an interval of 1 ps and the resulting set of configurations were recomputed with density functional theory using the PBE exchange-correlation functional with D3 dispersion correction and def2-SVP basis set and VeryTightSCF convergence settings using the ORCA electronic structure package."}],["DS_ztlvjmuwskom_0",{"n":"BOTnet_ACAC_2022_H_transfer","d":"NEB path of proton transfer reaction between the two forms of acetylacetone. Acetylacetone dataset generated from a long molecular dynamics simulation at 300 K using a Langevin thermostat at the semi-empirical GFN2-xTB level of theory. Configurations were sampled at an interval of 1 ps and the resulting set of configurations were recomputed with density functional theory using the PBE exchange-correlation functional with D3 dispersion correction and def2-SVP basis set and VeryTightSCF convergence settings using the ORCA electronic structure package."}],["DS_e22u3j5sbilx_0",{"n":"BOTnet_ACAC_2022_isolated","d":"Energies of the isolated atoms evalauted at the reference DFT settings. Acetylacetone dataset generated from a long molecular dynamics simulation at 300 K using a Langevin thermostat at the semi-empirical GFN2-xTB level of theory. Configurations were sampled at an interval of 1 ps and the resulting set of configurations were recomputed with density functional theory using the PBE exchange-correlation functional with D3 dispersion correction and def2-SVP basis set and VeryTightSCF convergence settings using the ORCA electronic structure package."}],["DS_gt0oa9fngod7_0",{"n":"BOTnet_ACAC_2022_test_300K_MD","d":"Test set of decorrelated geometries sampled from 300 K xTB MD. Acetylacetone dataset generated from a long molecular dynamics simulation at 300 K using a Langevin thermostat at the semi-empirical GFN2-xTB level of theory. Configurations were sampled at an interval of 1 ps and the resulting set of configurations were recomputed with density functional theory using the PBE exchange-correlation functional with D3 dispersion correction and def2-SVP basis set and VeryTightSCF convergence settings using the ORCA electronic structure package."}],["DS_k0rexhnv58vn_0",{"n":"BOTnet_ACAC_2022_test_600K_MD","d":"Test set of decorrelated geometries sampled from 600 K xTB MD. Acetylacetone dataset generated from a long molecular dynamics simulation at 300 K using a Langevin thermostat at the semi-empirical GFN2-xTB level of theory. Configurations were sampled at an interval of 1 ps and the resulting set of configurations were recomputed with density functional theory using the PBE exchange-correlation functional with D3 dispersion correction and def2-SVP basis set and VeryTightSCF convergence settings using the ORCA electronic structure package."}],["DS_8gqwz5bw1pvq_0",{"n":"BOTnet_ACAC_2022_train_300K_MD","d":"500 decorrelated geometries sampled from 300 K xTB MD run. Acetylacetone dataset generated from a long molecular dynamics simulation at 300 K using a Langevin thermostat at the semi-empirical GFN2-xTB level of theory. Configurations were sampled at an interval of 1 ps and the resulting set of configurations were recomputed with density functional theory using the PBE exchange-correlation functional with D3 dispersion correction and def2-SVP basis set and VeryTightSCF convergence settings using the ORCA electronic structure package."}],["DS_632yputkfhpg_0",{"n":"BOTnet_ACAC_2022_train_600K_MD","d":"500 decorrelated geometries sampled from 600 K xTB MD run. Acetylacetone dataset generated from a long molecular dynamics simulation at 300 K using a Langevin thermostat at the semi-empirical GFN2-xTB level of theory. Configurations were sampled at an interval of 1 ps and the resulting set of configurations were recomputed with density functional theory using the PBE exchange-correlation functional with D3 dispersion correction and def2-SVP basis set and VeryTightSCF convergence settings using the ORCA electronic structure package."}],["DS_olmzgsz9hmxy_0",{"n":"C7H10O2","d":"6095 isomers of C7O2H10. Energetics were calculated at the G4MP2 level of theory."}],["DS_l7inbtql4ea9_0",{"n":"CA-9_BB_training","d":"Binning-binning configurations from CA-9 dataset used for training NNP_BB potential. CA-9 consists of configurations of carbon with curated subsets chosen to test the effects of intentionally choosing dissimilar configurations when training neural network potentials"}],["DS_8zft35i5sbgd_0",{"n":"CA-9_BB_validation","d":"Binning-binning configurations from CA-9 dataset used during validation step for NNP_BB potential. CA-9 consists of configurations of carbon with curated subsets chosen to test the effects of intentionally choosing dissimilar configurations when training neural network potentials"}],["DS_fw7m5d8b0fa9_0",{"n":"CA-9_BR_training","d":"Binning-random configurations from CA-9 dataset used for training NNP_BR potential. CA-9 consists of configurations of carbon with curated subsets chosen to test the effects of intentionally choosing dissimilar configurations when training neural network potentials"}],["DS_fgakfcwhn9zc_0",{"n":"CA-9_BR_validation","d":"Binning-random configurations from CA-9 dataset used during validation step for NNP_BR potential. CA-9 consists of configurations of carbon with curated subsets chosen to test the effects of intentionally choosing dissimilar configurations when training neural network potentials"}],["DS_tag5zubl21w8_0",{"n":"CA-9_RR_training","d":"Random-random configurations from CA-9 dataset used for training NNP_RR potential. CA-9 consists of configurations of carbon with curated subsets chosen to test the effects of intentionally choosing dissimilar configurations when training neural network potentials"}],["DS_mr2qaiqh2sx2_0",{"n":"CA-9_RR_validation","d":"Random-random configurations from CA-9 dataset used during validation step for NNP_RR potential. CA-9 consists of configurations of carbon with curated subsets chosen to test the effects of intentionally choosing dissimilar configurations when training neural network potentials"}],["DS_h0mshvvbxlai_0",{"n":"CA-9_test","d":"Test configurations from CA-9 dataset used to evaluate trained NNPs.CA-9 consists of configurations of carbon with curated subsets chosen to test the effects of intentionally choosing dissimilar configurations when training neural network potentials"}],["DS_9cdtww9pcjqd_0",{"n":"CA-9_training","d":"Configurations from CA-9 dataset used for training NNP_CA-9 potential. CA-9 consists of configurations of carbon with curated subsets chosen to test the effects of intentionally choosing dissimilar configurations when training neural network potentials"}],["DS_07zjrtacjg3z_0",{"n":"CA-9_validation","d":"Configurations from CA-9 dataset used during validation step for NNP_CA-9 potential. CA-9 consists of configurations of carbon with curated subsets chosen to test the effects of intentionally choosing dissimilar configurations when training neural network potentials"}],["DS_392q5d027yja_0",{"n":"CGM-MLP_natcomm2023_Cr-C_deposition","d":"Training simulations from CGM-MLP_natcomm2023 of carbon deposition on a Cr surface. This dataset was one of the datasets used in training during the process of producing an active learning dataset for the purposes of exploring substrate-catalyzed deposition on metal surfaces such as Cu(111), Cr(110), Ti(001), and oxygen-contaminated Cu(111) as a means of controllable synthesis of carbon nanomaterials. The combined dataset includes structures from the Carbon_GAP_20 dataset and additional configurations of carbon clusters on a Cu, Cr and Ti surfaces."}],["DS_xho213jy5pf9_0",{"n":"CGM-MLP_natcomm2023_Cu-C-O","d":"Training simulations from CGM-MLP_natcomm2023 of carbon on an oxygen-contaminated Cu surface. This dataset was one of the datasets used in training during the process of producing an active learning dataset for the purposes of exploring substrate-catalyzed deposition on metal surfaces such as Cu(111), Cr(110), Ti(001), and oxygen-contaminated Cu(111) as a means of controllable synthesis of carbon nanomaterials. The combined dataset includes structures from the Carbon_GAP_20 dataset and additional configurations of carbon clusters on a Cu, Cr and Ti surfaces."}],["DS_xzo1tvni8bay_0",{"n":"CGM-MLP_natcomm2023_Cu-C-O_deposition","d":"Training simulations from CGM-MLP_natcomm2023 of carbon deposition on a Cu surface. This appears similar to CGM-MLP_natcomm2023_CU-C_deposition, as there are no O atoms present in this set. This dataset was one of the datasets used in training during the process of producing an active learning dataset for the purposes of exploring substrate-catalyzed deposition on metal surfaces such as Cu(111), Cr(110), Ti(001), and oxygen-contaminated Cu(111) as a means of controllable synthesis of carbon nanomaterials. The combined dataset includes structures from the Carbon_GAP_20 dataset and additional configurations of carbon clusters on a Cu, Cr and Ti surfaces."}],["DS_vgy50b4qz4p7_0",{"n":"CGM-MLP_natcomm2023_Cu-C_deposition","d":"Training simulations from CGM-MLP_natcomm2023 of carbon deposition on a Cu surface. This dataset was one of the datasets used in training during the process of producing an active learning dataset for the purposes of exploring substrate-catalyzed deposition on metal surfaces such as Cu(111), Cr(110), Ti(001), and oxygen-contaminated Cu(111) as a means of controllable synthesis of carbon nanomaterials. The combined dataset includes structures from the Carbon_GAP_20 dataset and additional configurations of carbon clusters on a Cu, Cr and Ti surfaces."}],["DS_mo7q9ajg1lw8_0",{"n":"CGM-MLP_natcomm2023_Cu-C_metal_surface","d":"Training simulations from CGM-MLP_natcomm2023 of carbon on a Cu metal surface. This dataset was one of the datasets used in training during the process of producing an active learning dataset for the purposes of exploring substrate-catalyzed deposition on metal surfaces such as Cu(111), Cr(110), Ti(001), and oxygen-contaminated Cu(111) as a means of controllable synthesis of carbon nanomaterials. The combined dataset includes structures from the Carbon_GAP_20 dataset and additional configurations of carbon clusters on a Cu, Cr and Ti surfaces."}],["DS_qslbmhe4bcgg_0",{"n":"CGM-MLP_natcomm2023_GAP_20","d":"Carbon_GAP_20 dataset from CGM-MLP_natcomm2023. This dataset was one of the datasets used in training during the process of producing an active learning dataset for the purposes of exploring substrate-catalyzed deposition on metal surfaces such as Cu(111), Cr(110), Ti(001), and oxygen-contaminated Cu(111) as a means of controllable synthesis of carbon nanomaterials. The combined dataset includes structures from the Carbon_GAP_20 dataset and additional configurations of carbon clusters on a Cu, Cr and Ti surfaces."}],["DS_8frq3jczb0dt_0",{"n":"CGM-MLP_natcomm2023_Ti-C_deposition","d":"Training simulations from CGM-MLP_natcomm2023 of carbon deposition on a Ti surface. This dataset was one of the datasets used in training during the process of producing an active learning dataset for the purposes of exploring substrate-catalyzed deposition on metal surfaces such as Cu(111), Cr(110), Ti(001), and oxygen-contaminated Cu(111) as a means of controllable synthesis of carbon nanomaterials. The combined dataset includes structures from the Carbon_GAP_20 dataset and additional configurations of carbon clusters on a Cu, Cr and Ti surfaces."}],["DS_wt0m3lxcdy52_0",{"n":"CGM-MLP_natcomm2023_screening_amorphous_carbon_test","d":"493 structures available from the GAP-20 database, excluding any structures present in the training set. This dataset was one of the datasets used in testing screening parameters during the process of producing an active learning dataset for Cu-C interactions for the purposes of exploring substrate-catalyzed deposition as a means of controllable synthesis of carbon nanomaterials. The combined dataset includes structures from the Carbon_GAP_20 dataset and additional configurations of carbon clusters on a Cu(111) surface."}],["DS_taxwl7jo4f8a_0",{"n":"CGM-MLP_natcomm2023_screening_amorphous_carbon_train","d":"2558 structures selected from the GAP-20 database. This dataset was one of the datasets used in testing screening parameters during the process of producing an active learning dataset for Cu-C interactions for the purposes of exploring substrate-catalyzed deposition as a means of controllable synthesis of carbon nanomaterials. The combined dataset includes structures from the Carbon_GAP_20 dataset and additional configurations of carbon clusters on a Cu(111) surface."}],["DS_gkumjkncy8ft_0",{"n":"CGM-MLP_natcomm2023_screening_carbon-cluster@Cu_test","d":"192 structures were uniformly selected from the AIMD simulation, excluding any structures that are part of the training set. This dataset was one of the datasets used in testing screening parameters during the process of producing an active learning dataset for Cu-C interactions for the purposes of exploring substrate-catalyzed deposition as a means of controllable synthesis of carbon nanomaterials. The combined dataset includes structures from the Carbon_GAP_20 dataset and additional configurations of carbon clusters on a Cu(111) surface."}],["DS_6sjhg1f8fv5j_0",{"n":"CGM-MLP_natcomm2023_screening_carbon-cluster@Cu_train","d":"588 structures selected from the AIMD simulation of the Cu(111) slab, including both the C1-C18 clusters on the Cu(111) slab. This dataset was one of the datasets used in testing screening parameters during the process of producing an active learning dataset for Cu-C interactions for the purposes of exploring substrate-catalyzed deposition as a means of controllable synthesis of carbon nanomaterials. The combined dataset includes structures from the Carbon_GAP_20 dataset and additional configurations of carbon clusters on a Cu(111) surface."}],["DS_5r9dvov8nq7c_0",{"n":"CGM-MLP_natcomm2023_screening_deposited-carbon@Cu_test","d":"468 structures uniformly selected from the MD/tfMC simulation, excluding any structures that are part of the training set. This dataset was one of the datasets used in testing screening parameters during the process of producing an active learning dataset for Cu-C interactions for the purposes of exploring substrate-catalyzed deposition as a means of controllable synthesis of carbon nanomaterials. The combined dataset includes structures from the Carbon_GAP_20 dataset and additional configurations of carbon clusters on a Cu(111) surface."}],["DS_j6w4ru800ukq_0",{"n":"CGM-MLP_natcomm2023_screening_deposited-carbon@Cu_train","d":"1090 structures uniformly selected from the MD/tfMC simulation during the training process of CGM-MLPs. This dataset was one of the datasets used in testing screening parameters during the process of producing an active learning dataset for Cu-C interactions for the purposes of exploring substrate-catalyzed deposition as a means of controllable synthesis of carbon nanomaterials. The combined dataset includes structures from the Carbon_GAP_20 dataset and additional configurations of carbon clusters on a Cu(111) surface."}],["DS_jasbxoigo7r4_0",{"n":"CGM-MLP_natcomm2023_screening_graphite_train","d":"40 graphite structures with different lattice constants ranging from 2.0 to 3.2 \u00c5, with a 0.03 \u00c5 increment. This dataset was one of the datasets used in testing screening parameters during the process of producing an active learning dataset for Cu-C interactions for the purposes of exploring substrate-catalyzed deposition as a means of controllable synthesis of carbon nanomaterials. The combined dataset includes structures from the Carbon_GAP_20 dataset and additional configurations of carbon clusters on a Cu(111) surface."}],["DS_3td9plyix4ib_0",{"n":"CHON_JCP_2020","d":"This dataset of molecular structures was extracted, using the NOMAD API, from all available structures in the NOMAD Archive that only include C, H, O, and N. This dataset consists of 50.42% H, 30.41% C, 10.36% N, and 8.81% O and includes 96 804 atomic environments in 5217 structures."}],["DS_l9f0rjjqfd67_0",{"n":"COHInPt_schaaf_2023","d":"Training and simulation data from machine learning force field model applied to steps of the hydrogenation of carbon dioxide to methanol over an indium oxide catalyst, with and without platinum doping."}],["DS_anwiyxoek66t_0",{"n":"COLL_test","d":"Test set from COLL. Consists of configurations taken from molecular collisions of different small organic molecules. Energies and forces for 140,000 random snapshots taken from these trajectories were recomputed with density functional theory (DFT). These calculations were performed with the revPBE functional and def2-TZVP basis, including D3 dispersion corrections"}],["DS_cifhpgzw3ckj_0",{"n":"COLL_train","d":"Training set from COLL. Consists of configurations taken from molecular collisions of different small organic molecules. Energies and forces for 140,000 random snapshots taken from these trajectories were recomputed with density functional theory (DFT). These calculations were performed with the revPBE functional and def2-TZVP basis, including D3 dispersion corrections"}],["DS_1y25o4zvyfm0_0",{"n":"COLL_validation","d":"Validation set from COLL. Consists of configurations taken from molecular collisions of different small organic molecules. Energies and forces for 140,000 random snapshots taken from these trajectories were recomputed with density functional theory (DFT). These calculations were performed with the revPBE functional and def2-TZVP basis, including D3 dispersion corrections"}],["DS_0dbwoxq96gga_0",{"n":"COMP6v2-B973c-def2mTZVP","d":"COMP6v2-B973c-def2mTZVP is the portion of COMP6v2 calculated at the B973c/def2mTZVP level of theory. COmprehensive Machine-learning Potential (COMP6) Benchmark Suite version 2.0 is an extension of the COMP6 benchmark found in the following repository: https://github.com/isayev/COMP6. COMP6v2 is a data set of density functional properties for molecules containing H, C, N, O, S, F, and Cl. It is available at the following levels of theory: wB97X/631Gd (data used to train model in the ANI-2x paper); wB97MD3BJ/def2TZVPP; wB97MV/def2TZVPP; B973c/def2mTZVP. The 6 subsets from COMP6 (ANI-MD, DrugBank, GDB07to09, GDB10to13 Tripeptides, and s66x8) are contained in each of the COMP6v2 datasets corresponding to the above levels of theory."}],["DS_m6etjqhlu40m_0",{"n":"COMP6v2-wB97MV-def2TZVPP","d":"COMP6v2-wB97MV-def2TZVPP is the portion of COMP6v2 calculated at the wB97MV/def2TZVPP level of theory. COmprehensive Machine-learning Potential (COMP6) Benchmark Suite version 2.0 is an extension of the COMP6 benchmark found in the following repository: https://github.com/isayev/COMP6. COMP6v2 is a data set of density functional properties for molecules containing H, C, N, O, S, F, and Cl. It is available at the following levels of theory: wB97X/631Gd (data used to train model in the ANI-2x paper); wB97MD3BJ/def2TZVPP; wB97MV/def2TZVPP; B973c/def2mTZVP. The 6 subsets from COMP6 (ANI-MD, DrugBank, GDB07to09, GDB10to13 Tripeptides, and s66x8) are contained in each of the COMP6v2 datasets corresponding to the above levels of theory."}],["DS_15m7etw1om8a_0",{"n":"COMP6v2-wB97X-631Gd","d":"COMP6v2-wB97X-631Gd is the portion of COMP6v2 calculated at the wB97X/631Gd level of theory. COmprehensive Machine-learning Potential (COMP6) Benchmark Suite version 2.0 is an extension of the COMP6 benchmark found in the following repository: https://github.com/isayev/COMP6. COMP6v2 is a data set of density functional properties for molecules containing H, C, N, O, S, F, and Cl. It is available at the following levels of theory: wB97X/631Gd (data used to train model in the ANI-2x paper); wB97MD3BJ/def2TZVPP; wB97MV/def2TZVPP; B973c/def2mTZVP. The 6 subsets from COMP6 (ANI-MD, DrugBank, GDB07to09, GDB10to13 Tripeptides, and s66x8) are contained in each of the COMP6v2 datasets corresponding to the above levels of theory."}],["DS_mma0w04wa4cs_0",{"n":"C_Gardner_2022","d":"Approximately 115,000 configurations of carbon with 200 atoms, with simulated melt, quench, reheat, then annealing at the noted temperature. Includes a variety of carbon structures."}],["DS_qph0akhjv9kv_0",{"n":"C_NPJ2020","d":"The dataset consists of energies and forces for monolayer graphene, bilayer graphene, graphite, and diamond in various states, including strained static structures and configurations drawn from ab initio MD trajectories. A total number of 4788 configurations was generated from DFT calculations using the Vienna Ab initio Simulation Package (VASP). The energies and forces are stored in the extended XYZ format. One file for each configuration."}],["DS_on6fusdw5n8q_0",{"n":"Carbon_GAP_JCP_2020","d":"GAP-20 describes the properties of the bulk crystalline and amorphous phases, crystal surfaces, and defect structures with an accuracy approaching that of direct ab initio simulation, but at a significantly reduced cost. The final potential is fitted to reference data computed using the optB88-vdW density functional theory (DFT) functional."}],["DS_sz9me9r61581_0",{"n":"Carbon_GAP_JCP_2020_train","d":"Training data generated for GAP-20. GAP-20 describes the properties of the bulk crystalline and amorphous phases, crystal surfaces, and defect structures with an accuracy approaching that of direct ab initio simulation, but at a significantly reduced cost. The final potential is fitted to reference data computed using the optB88-vdW density functional theory (DFT) functional."}],["DS_r2r8k3fyb6ny_0",{"n":"Carolina_Materials","d":"Carolina Materials contains structures used to train several machine learning models for the efficient generation of hypothetical inorganic materials. The database is built using structures from OQMD, Materials Project and ICSD, as well as ML generated structures validated by DFT. "}],["DS_y6bp7td4dle0_0",{"n":"Cationic_phenoxyimine_complexes_of_yttrium","d":"This dataset contains DFT calculations that were carried out in conjunction with experimental investigation of a cationic phenoxyimine yttrium complex as an isoprene polimerization catalyst. Calculations were performed using the Gaussian 09 D.01 suite of programs.Electronic structure calculations were performed at the DFT level using the B3PW91 functional. The Stuttgart-Cologne small-core quasi-relativistic pseudopotential ECP28MWB and its available basis set including up to the g function were used to describe yttrium. Similarly, silicon and phosphorus were represented by a Stuttgart-Dresden-Bonn pseudopotential along with the related basis set augmented by a d function of polarization (\u03b1d(P) = 0.387 and \u03b1d(Si) = 0.284). Other atoms were described by a polarized all-electron triple-\u03b6 6-311G(d,p) basis set. Bulk solvent effect of toluene or THF was simulated using the SMD continuum model. The Grimme empirical correction with the original D3 damping function was used to include the dispersion correction as a single-point calculation. Transition-state optimization was followed by frequency calculations to characterize the stationary point. Intrinsic reaction coordinate calculations were performed to confirm the connectivity of the transition states. Gibbs energies were estimated within the harmonic oscillator approximation and estimated at 298 K and 1 atm."}],["DS_e08dhew7z0r6_0",{"n":"Chig-AIMD_random_test","d":"Test configurations from the 'random' split of Chig-AIMD. This dataset covers the conformational space of chignolin with DFT-level precision. We sequentially applied replica exchange molecular dynamics (REMD), conventional MD, and ab initio MD (AIMD) simulations on a 10 amino acid protein, Chignolin, and finally collected 2 million biomolecule structures with quantum level energy and force records."}],["DS_dsikv10na4f8_0",{"n":"Chig-AIMD_random_train","d":"Training configurations from the 'random' split of Chig-AIMD. This dataset covers the conformational space of chignolin with DFT-level precision. We sequentially applied replica exchange molecular dynamics (REMD), conventional MD, and ab initio MD (AIMD) simulations on a 10 amino acid protein, Chignolin, and finally collected 2 million biomolecule structures with quantum level energy and force records."}],["DS_u0ggh1iie0uc_0",{"n":"Chig-AIMD_random_val","d":"Validation configurations from the 'random' split of Chig-AIMD. This dataset covers the conformational space of chignolin with DFT-level precision. We sequentially applied replica exchange molecular dynamics (REMD), conventional MD, and ab initio MD (AIMD) simulations on a 10 amino acid protein, Chignolin, and finally collected 2 million biomolecule structures with quantum level energy and force records."}],["DS_zmzkyvep8l55_0",{"n":"Chig-AIMD_scaffold_test","d":"Test configurations from the 'scaffold' split of Chig-AIMD. This dataset covers the conformational space of chignolin with DFT-level precision. We sequentially applied replica exchange molecular dynamics (REMD), conventional MD, and ab initio MD (AIMD) simulations on a 10 amino acid protein, Chignolin, and finally collected 2 million biomolecule structures with quantum level energy and force records."}],["DS_7puixss6qd61_0",{"n":"Chig-AIMD_scaffold_train","d":"Training configurations from the 'scaffold' split of Chig-AIMD. This dataset covers the conformational space of chignolin with DFT-level precision. We sequentially applied replica exchange molecular dynamics (REMD), conventional MD, and ab initio MD (AIMD) simulations on a 10 amino acid protein, Chignolin, and finally collected 2 million biomolecule structures with quantum level energy and force records."}],["DS_mzz13lim5qfi_0",{"n":"Chig-AIMD_scaffold_val","d":"Validation configurations from the 'scaffold' split of Chig-AIMD. This dataset covers the conformational space of chignolin with DFT-level precision. We sequentially applied replica exchange molecular dynamics (REMD), conventional MD, and ab initio MD (AIMD) simulations on a 10 amino acid protein, Chignolin, and finally collected 2 million biomolecule structures with quantum level energy and force records."}],["DS_3a3auhtmkv45_0",{"n":"Co-Co_coupling_at_liquid_water-Cu(100)_interfaces_JC2021","d":"This dataset contains data from eight AIMD simulations run in VASP to study electrochemical *CO-*CO coupling -- coupling of two *CO molecules -- at the liquid water-Cu(100) interface."}],["DS_4w9gf8dxzskh_0",{"n":"CoCrFeNiPd_MRL2020","d":"The dataset for \"Origin of high strength in the CoCrFeNiPd high-entropy alloy\", containing DFT-calculated values of the high-entropy alloy CoCrFeNiPd, created to explore the reasons behind experimental findings of the increased strength CoCrFeNiPd in comparison to CoCrFeNi."}],["DS_sn623uhg2d1b_0",{"n":"CoNbV_CMS2019","d":"This dataset was generated using the following active learning scheme: 1) candidate structures were relaxed by a partially-trained MTP model, 2) structures for which the MTP had to perform extrapolation were passed to DFT to be re-computed, 3) the MTP was retrained, including the structures that were re-computed with DFT, 4) steps 1-3 were repeated until the MTP no longer extrapolated on any of the original candidate structures. The original candidate structures for this dataset included about 27,000 configurations that were bcc-like and close-packed (fcc, hcp, etc.) with 8 or fewer atoms in the unit cell and different concentrations of Co, Nb, and V."}],["DS_uwqpfuuj6utw_0",{"n":"Co_dimer_JPCA_2022","d":"This dataset contains dimer molecules of Co(II) with potential energy calculations for structures with ferromagnetic and antiferromagnetic spin configurations. Calculations were carried out in Gaussian 16 with the PBE exchange-correlation functional and 6-31+G* basis set. All molecules contain the same atomic core region, consisting of the tetrahedral and octahedral Co centers and the three PO2R2 bridging ligands. The ligand exchange provides a broad range of exchange energies (\u0394EJ), from +50 to -200 meV, with 80% of the ligands yielding \u0394EJ < 10 meV."}],["DS_doo9ltj3vsnu_0",{"n":"Co_dimer_JPCA_2022_train","d":"Training data only from the Co_dimer_JPCA_2022 dataset. This dataset contains dimer molecules of Co(II) with potential energy calculations for structures with ferromagnetic and antiferromagnetic spin configurations. Calculations were carried out in Gaussian 16 with the PBE exchange-correlation functional and 6-31+G* basis set. All molecules contain the same atomic core region, consisting of the tetrahedral and octahedral Co centers and the three PO2R2 bridging ligands. The ligand exchange provides a broad range of exchange energies (\u0394EJ), from +50 to -200 meV, with 80% of the ligands yielding \u0394EJ < 10 meV."}],["DS_xs942mj0c3dx_0",{"n":"ComBat","d":"DFT-optimized geometries and properties for Li-S electrolytes. These make up the Computational Database for Li-S Batteries (ComBat), calculated using Gaussian 16 at the B3LYP/6-31+G* level of theory."}],["DS_0ry6z1j8mi8c_0",{"n":"CuPd_CMS2019","d":"This dataset was generated using the following active learning scheme: 1) candidate structures were relaxed by a partially-trained MTP model, 2) structures for which the MTP had to perform extrapolation were passed to DFT to be re-computed, 3) the MTP was retrained, including the structures that were re-computed with DFT, 4) steps 1-3 were repeated until the MTP no longer extrapolated on any of the original candidate structures. The original candidate structures for this dataset included 40,000 unrelaxed configurations with BCC, FCC, and HCP lattices."}],["DS_39v8vn61tlzl_0",{"n":"Cu_FHI-aims_NPJCM_2021","d":"Approximately 46,000 configurations of copper, including small and bulk structures, surfaces, interfaces, point defects, and randomly modified variants. Also includes structures with displaced or missing atoms."}],["DS_eyo7scxvrfq0_0",{"n":"DAS_MLIP_CoSb_MgSb","d":"Approximately 850 configurations of CoSb3 and Mg3Sb2 generated using a dual adaptive sampling (DAS) method for use with machine learning of interatomic potentials (MLIP)."}],["DS_ech19sh1koj4_0",{"n":"DFT_polymorphs_PNAS_2022_PBE0_MBD_benzene_test","d":"Benzene test PBE0-MBD dataset from \"Semi-local and hybrid functional DFT data for thermalised snapshots of polymorphs of benzene, succinic acid, and glycine\". DFT reference energies and forces were calculated using Quantum Espresso v6.3. The calculations were performed with the semi-local PBE xc functional, Tkatchenko-Scheffler dispersion correction, optimised norm-conserving Vanderbilt pseudopotentials, a Monkhorst-Pack k-point grid with a maximum spacing of 0.06 x 2\u03c0 A^-1, and a plane-wave energy cut-off of 100 Ry for the wavefunction."}],["DS_ce4zytu7ph7e_0",{"n":"DFT_polymorphs_PNAS_2022_PBE0_MBD_benzene_train","d":"Benzene training PBE0-MBD dataset from \"Semi-local and hybrid functional DFT data for thermalised snapshots of polymorphs of benzene, succinic acid, and glycine\". DFT reference energies and forces were calculated using Quantum Espresso v6.3. The calculations were performed with the semi-local PBE xc functional, Tkatchenko-Scheffler dispersion correction, optimised norm-conserving Vanderbilt pseudopotentials, a Monkhorst-Pack k-point grid with a maximum spacing of 0.06 x 2\u03c0 A^-1, and a plane-wave energy cut-off of 100 Ry for the wavefunction."}],["DS_xaq6jwha0a7o_0",{"n":"DFT_polymorphs_PNAS_2022_PBE0_MBD_benzene_validation","d":"Benzene validation PBE0-MBD dataset from \"Semi-local and hybrid functional DFT data for thermalised snapshots of polymorphs of benzene, succinic acid, and glycine\". DFT reference energies and forces were calculated using Quantum Espresso v6.3. The calculations were performed with the semi-local PBE xc functional, Tkatchenko-Scheffler dispersion correction, optimised norm-conserving Vanderbilt pseudopotentials, a Monkhorst-Pack k-point grid with a maximum spacing of 0.06 x 2\u03c0 A^-1, and a plane-wave energy cut-off of 100 Ry for the wavefunction."}],["DS_h3jymhr74qv4_0",{"n":"DFT_polymorphs_PNAS_2022_PBE0_MBD_glycine_test","d":"Glycine test PBE0-MBD dataset from \"Semi-local and hybrid functional DFT data for thermalised snapshots of polymorphs of benzene, succinic acid, and glycine\". DFT reference energies and forces were calculated using Quantum Espresso v6.3. The calculations were performed with the semi-local PBE xc functional, Tkatchenko-Scheffler dispersion correction, optimised norm-conserving Vanderbilt pseudopotentials, a Monkhorst-Pack k-point grid with a maximum spacing of 0.06 x 2\u03c0 A^-1, and a plane-wave energy cut-off of 100 Ry for the wavefunction."}],["DS_b1n228x610dj_0",{"n":"DFT_polymorphs_PNAS_2022_PBE0_MBD_glycine_train","d":"Glycine training PBE0-MBD dataset from \"Semi-local and hybrid functional DFT data for thermalised snapshots of polymorphs of benzene, succinic acid, and glycine\". DFT reference energies and forces were calculated using Quantum Espresso v6.3. The calculations were performed with the semi-local PBE xc functional, Tkatchenko-Scheffler dispersion correction, optimised norm-conserving Vanderbilt pseudopotentials, a Monkhorst-Pack k-point grid with a maximum spacing of 0.06 x 2\u03c0 A^-1, and a plane-wave energy cut-off of 100 Ry for the wavefunction."}],["DS_n1nilf7ralhq_0",{"n":"DFT_polymorphs_PNAS_2022_PBE0_MBD_glycine_validation","d":"Glycine validation PBE0-MBD dataset from \"Semi-local and hybrid functional DFT data for thermalised snapshots of polymorphs of benzene, succinic acid, and glycine\". DFT reference energies and forces were calculated using Quantum Espresso v6.3. The calculations were performed with the semi-local PBE xc functional, Tkatchenko-Scheffler dispersion correction, optimised norm-conserving Vanderbilt pseudopotentials, a Monkhorst-Pack k-point grid with a maximum spacing of 0.06 x 2\u03c0 A^-1, and a plane-wave energy cut-off of 100 Ry for the wavefunction."}],["DS_vbjvssevrff5_0",{"n":"DFT_polymorphs_PNAS_2022_PBE0_MBD_succinic_acid_test","d":"Succinic acid test PBE0-MBD dataset from \"Semi-local and hybrid functional DFT data for thermalised snapshots of polymorphs of benzene, succinic acid, and glycine\". DFT reference energies and forces were calculated using Quantum Espresso v6.3. The calculations were performed with the semi-local PBE xc functional, Tkatchenko-Scheffler dispersion correction, optimised norm-conserving Vanderbilt pseudopotentials, a Monkhorst-Pack k-point grid with a maximum spacing of 0.06 x 2\u03c0 A^-1, and a plane-wave energy cut-off of 100 Ry for the wavefunction."}],["DS_pqmtyp5bxd46_0",{"n":"DFT_polymorphs_PNAS_2022_PBE0_MBD_succinic_acid_train","d":"Succinic acid training PBE0-MBD dataset from \"Semi-local and hybrid functional DFT data for thermalised snapshots of polymorphs of benzene, succinic acid, and glycine\". DFT reference energies and forces were calculated using Quantum Espresso v6.3. The calculations were performed with the semi-local PBE xc functional, Tkatchenko-Scheffler dispersion correction, optimised norm-conserving Vanderbilt pseudopotentials, a Monkhorst-Pack k-point grid with a maximum spacing of 0.06 x 2\u03c0 A^-1, and a plane-wave energy cut-off of 100 Ry for the wavefunction."}],["DS_w18131j6r4oo_0",{"n":"DFT_polymorphs_PNAS_2022_PBE0_MBD_succinic_acid_validation","d":"Succinic acid validation PBE0-MBD dataset from \"Semi-local and hybrid functional DFT data for thermalised snapshots of polymorphs of benzene, succinic acid, and glycine\". DFT reference energies and forces were calculated using Quantum Espresso v6.3. The calculations were performed with the semi-local PBE xc functional, Tkatchenko-Scheffler dispersion correction, optimised norm-conserving Vanderbilt pseudopotentials, a Monkhorst-Pack k-point grid with a maximum spacing of 0.06 x 2\u03c0 A^-1, and a plane-wave energy cut-off of 100 Ry for the wavefunction."}],["DS_23820czfaq24_0",{"n":"DFT_polymorphs_PNAS_2022_PBE_TS_benzene_test","d":"Benzene test PBE-TS dataset from \"Semi-local and hybrid functional DFT data for thermalised snapshots of polymorphs of benzene, succinic acid, and glycine\". DFT reference energies and forces were calculated using Quantum Espresso v6.3. The calculations were performed with the semi-local PBE xc functional, Tkatchenko-Scheffler dispersion correction, optimised norm-conserving Vanderbilt pseudopotentials, a Monkhorst-Pack k-point grid with a maximum spacing of 0.06 x 2\u03c0 A^-1, and a plane-wave energy cut-off of 100 Ry for the wavefunction."}],["DS_3qi25f3sxkwr_0",{"n":"DFT_polymorphs_PNAS_2022_PBE_TS_benzene_train","d":"Benzene training PBE-TS dataset from \"Semi-local and hybrid functional DFT data for thermalised snapshots of polymorphs of benzene, succinic acid, and glycine\". DFT reference energies and forces were calculated using Quantum Espresso v6.3. The calculations were performed with the semi-local PBE xc functional, Tkatchenko-Scheffler dispersion correction, optimised norm-conserving Vanderbilt pseudopotentials, a Monkhorst-Pack k-point grid with a maximum spacing of 0.06 x 2\u03c0 A^-1, and a plane-wave energy cut-off of 100 Ry for the wavefunction."}],["DS_8ou78o18mvfz_0",{"n":"DFT_polymorphs_PNAS_2022_PBE_TS_benzene_validation","d":"Benzene validation PBE-TS dataset from \"Semi-local and hybrid functional DFT data for thermalised snapshots of polymorphs of benzene, succinic acid, and glycine\". DFT reference energies and forces were calculated using Quantum Espresso v6.3. The calculations were performed with the semi-local PBE xc functional, Tkatchenko-Scheffler dispersion correction, optimised norm-conserving Vanderbilt pseudopotentials, a Monkhorst-Pack k-point grid with a maximum spacing of 0.06 x 2\u03c0 A^-1, and a plane-wave energy cut-off of 100 Ry for the wavefunction."}],["DS_qx4h8j4luk70_0",{"n":"DFT_polymorphs_PNAS_2022_PBE_TS_glycine_test","d":"Glycine test PBE-TS dataset from \"Semi-local and hybrid functional DFT data for thermalised snapshots of polymorphs of benzene, succinic acid, and glycine\". DFT reference energies and forces were calculated using Quantum Espresso v6.3. The calculations were performed with the semi-local PBE xc functional, Tkatchenko-Scheffler dispersion correction, optimised norm-conserving Vanderbilt pseudopotentials, a Monkhorst-Pack k-point grid with a maximum spacing of 0.06 x 2\u03c0 A^-1, and a plane-wave energy cut-off of 100 Ry for the wavefunction."}],["DS_op9kvcm7ui6l_0",{"n":"DFT_polymorphs_PNAS_2022_PBE_TS_glycine_train","d":"Glycine training PBE-TS dataset from \"Semi-local and hybrid functional DFT data for thermalised snapshots of polymorphs of benzene, succinic acid, and glycine\". DFT reference energies and forces were calculated using Quantum Espresso v6.3. The calculations were performed with the semi-local PBE xc functional, Tkatchenko-Scheffler dispersion correction, optimised norm-conserving Vanderbilt pseudopotentials, a Monkhorst-Pack k-point grid with a maximum spacing of 0.06 x 2\u03c0 A^-1, and a plane-wave energy cut-off of 100 Ry for the wavefunction."}],["DS_052bb0zufjaz_0",{"n":"DFT_polymorphs_PNAS_2022_PBE_TS_glycine_validation","d":"Glycine validation PBE-TS dataset from \"Semi-local and hybrid functional DFT data for thermalised snapshots of polymorphs of benzene, succinic acid, and glycine\". DFT reference energies and forces were calculated using Quantum Espresso v6.3. The calculations were performed with the semi-local PBE xc functional, Tkatchenko-Scheffler dispersion correction, optimised norm-conserving Vanderbilt pseudopotentials, a Monkhorst-Pack k-point grid with a maximum spacing of 0.06 x 2\u03c0 A^-1, and a plane-wave energy cut-off of 100 Ry for the wavefunction."}],["DS_qmsg3kejmsy2_0",{"n":"DFT_polymorphs_PNAS_2022_PBE_TS_succinic_acid_test","d":"Succinic acid test PBE-TS dataset from \"Semi-local and hybrid functional DFT data for thermalised snapshots of polymorphs of benzene, succinic acid, and glycine\". DFT reference energies and forces were calculated using Quantum Espresso v6.3. The calculations were performed with the semi-local PBE xc functional, Tkatchenko-Scheffler dispersion correction, optimised norm-conserving Vanderbilt pseudopotentials, a Monkhorst-Pack k-point grid with a maximum spacing of 0.06 x 2\u03c0 A^-1, and a plane-wave energy cut-off of 100 Ry for the wavefunction."}],["DS_0u0k9ghrlkpx_0",{"n":"DFT_polymorphs_PNAS_2022_PBE_TS_succinic_acid_train","d":"Succinic acid training PBE-TS dataset from \"Semi-local and hybrid functional DFT data for thermalised snapshots of polymorphs of benzene, succinic acid, and glycine\". DFT reference energies and forces were calculated using Quantum Espresso v6.3. The calculations were performed with the semi-local PBE xc functional, Tkatchenko-Scheffler dispersion correction, optimised norm-conserving Vanderbilt pseudopotentials, a Monkhorst-Pack k-point grid with a maximum spacing of 0.06 x 2\u03c0 A^-1, and a plane-wave energy cut-off of 100 Ry for the wavefunction."}],["DS_nny0vpj1dy9s_0",{"n":"DFT_polymorphs_PNAS_2022_PBE_TS_succinic_acid_validation","d":"Succinic acid validation PBE-TS dataset from \"Semi-local and hybrid functional DFT data for thermalised snapshots of polymorphs of benzene, succinic acid, and glycine\". DFT reference energies and forces were calculated using Quantum Espresso v6.3. The calculations were performed with the semi-local PBE xc functional, Tkatchenko-Scheffler dispersion correction, optimised norm-conserving Vanderbilt pseudopotentials, a Monkhorst-Pack k-point grid with a maximum spacing of 0.06 x 2\u03c0 A^-1, and a plane-wave energy cut-off of 100 Ry for the wavefunction."}],["DS_101uk70asqhq_0",{"n":"DP-GEN_Cu","d":"Approximately 15,000 configurations of copper used to demonstrate the DP-GEN data generator for PES machine learning models."}],["DS_k85kj1kiekip_0",{"n":"DeePMD_SE","d":"127,000 configurations from a dataset used to benchmark and train a modified DeePMD model called DeepPot-SE, or Deep Potential - Smooth Edition"}],["DS_5h3810yhu4wj_0",{"n":"Fe_nanoparticles_PRB_2023","d":"This iron nanoparticles database contains dimers; trimers; bcc, fcc, hexagonal close-packed (hcp), simple cubic, and diamond crystalline structures. A wide range of cell parameters, as well as rattled structures, bcc-fcc and bcc-hcp transitional structures, surface slabs cleaved from relaxed bulk structures, nanoparticles and liquid configurations are included. The energy, forces and virials for the atomic structures were computed at the DFT level of theory using VASP with the PBE functional and standard PAW pseudopotentials for Fe (with 8 valence electrons, 4s^23d^6). The kinetic energy cutoff for plane waves was set to 400 eV and the energy threshold for convergence was 10-7 eV. All the DFT calculations were carried out with spin polarization."}],["DS_ej5u0tuycoph_0",{"n":"FitSNAP_Fe_NPJ_2021","d":"About 2,500 configurations of alpha-Fe used in the training and testing of a ML model with the goal of building magneto-elastic machine-learning interatomic potentials for large-scale spin-lattice dynamics simulations."}],["DS_tku3ae1rtxiy_0",{"n":"Forces_are_not_enough","d":"Approximately 300,000 benchmarking configurations derived partly from the MD-17 and LiPS datasets, partly from original simulated water and alanine dipeptide configurations."}],["DS_2s5grfmdez1q_0",{"n":"GDB_9_nature_2014","d":"133,855 configurations of stable small organic molecules composed of CHONF. A subset of GDB-17, with calculations of energies, dipole moment, polarizability and enthalpy. Calculations performed at B3LYP/6-31G(2df,p) level of theory."}],["DS_0x8ozlt9g3y5_0",{"n":"GFN-xTB_JCIM_2021","d":"10,000 configurations of organosilicon compounds with energies predicted by an improved GFN-xTB Hamiltonian parameterization, using revPBE."}],["DS_efcq7a0h6z5w_0",{"n":"GST_GAP_22_extended","d":"The extended training dataset for GST_GAP_22, calculated using the PBEsol functional. New configurations, simulated under external electric fields, were labelled with DFT and added to the original reference database GST-GAP-22 contains configurations of phase-change materials on the quasi-binary GeTe-Sb2Te3 (GST) line of chemical compositions. Data was used for training a machine learning interatomic potential to simulate a range of germanium-antimony-tellurium compositions under realistic device conditions."}],["DS_r3hav37ufnmb_0",{"n":"GST_GAP_22_main","d":"The main training dataset for GST_GAP_22, calculated using the PBEsol functional. GST-GAP-22 contains configurations of phase-change materials on the quasi-binary GeTe-Sb2Te3 (GST) line of chemical compositions. Data was used for training a machine learning interatomic potential to simulate a range of germanium-antimony-tellurium compositions under realistic device conditions."}],["DS_jy3ylaf48xg3_0",{"n":"GST_GAP_22_refitted","d":"The training dataset for GST_GAP_22, recalculated using the PBE functional. GST-GAP-22 contains configurations of phase-change materials on the quasi-binary GeTe-Sb2Te3 (GST) line of chemical compositions. Data was used for training a machine learning interatomic potential to simulate a range of germanium-antimony-tellurium compositions under realistic device conditions."}],["DS_in2572v33lon_0",{"n":"HDNNP_H2O","d":"Approximately 28,000 configurations split into 4 datasets, each using a different functional, used in the training of a high-dimensional neural network potential (HDNNP). "}],["DS_d5zht3ykr8hi_0",{"n":"HEA25S_high_entropy_alloys","d":"Dataset from \"Surface segregation in high-entropy alloys from alchemical machine learning: dataset HEA25S\". Includes 10000 bulk HEA structures (Dataset O), 2640 HEA surface slabs (Dataset A), together with 1000 bulk and 1000 surface slabs snapshots from the molecular dynamics (MD) runs (Datasets B and C), and 500 MD snapshots of the 25 elements Cantor-style alloy surface slabs. These splits, along with their respective train, test, and validation splits, are included as configuration sets."}],["DS_hvx1spm3gmia_0",{"n":"HEA25_high_entropy_transition-metal_alloys","d":"Dataset from \"Modeling high-entropy transition-metal alloys with alchemical compression\". Includes 25,000 structures utilized for fitting the aforementioned potential, with a focus on 25 d-block transition metals, excluding Tc, Cd, Re, Os and Hg. Each configuration includes a \"class\" field, indicating the crystal class of the structure. The class represents the following: 1: perfect crystals; 3-8 elements per structure, 2: shuffled positions (standard deviation 0.2\\AA ); 3-8 elements per structure, 3: shuffled positions (standard deviation 0.5\\AA ); 3-8 elements per structure, 4: shuffled positions (standard deviation 0.2\\AA ); 3-25 elements per structure. Configuration sets include divisions into fcc and bcc crystals, further split by class as described above."}],["DS_cpgyq72fs7uk_0",{"n":"HME21_test","d":"The test set from HME21. The  high-temperature multi-element 2021 (HME21) dataset comprises approximately 25,000 configurations, including 37 elements, used in the training of a universal NNP called PreFerential Potential (PFP). The dataset specifically contains disordered and unstable structures, and structures that include irregular substitutions, as well as varied temperature and density. "}],["DS_jhfis7syauhm_0",{"n":"HME21_train","d":"The training set from HME21. The  high-temperature multi-element 2021 (HME21) dataset comprises approximately 25,000 configurations, including 37 elements, used in the training of a universal NNP called PreFerential Potential (PFP). The dataset specifically contains disordered and unstable structures, and structures that include irregular substitutions, as well as varied temperature and density. "}],["DS_vf8hkwxhibio_0",{"n":"HME21_validation","d":"The validation set from HME21. The  high-temperature multi-element 2021 (HME21) dataset comprises approximately 25,000 configurations, including 37 elements, used in the training of a universal NNP called PreFerential Potential (PFP). The dataset specifically contains disordered and unstable structures, and structures that include irregular substitutions, as well as varied temperature and density. "}],["DS_1i4q8uyams32_0",{"n":"HO_LiMoNiTi_NPJCM_2020_LiMoNiTi_train","d":"Training configurations of Li8Mo2Ni7Ti7O32 from HO_LiMoNiTi_NPJCM_2020 used in the training of an ANN, whereby total energy is extrapolated by a Taylor expansion as a means of reducing computational costs."}],["DS_fgjil336jos6_0",{"n":"HO_LiMoNiTi_NPJCM_2020_LiMoNiTi_validation","d":"Validation configurations of Li8Mo2Ni7Ti7O32 from HO_LiMoNiTi_NPJCM_2020 used in the training of an ANN, whereby total energy is extrapolated by a Taylor expansion as a means of reducing computational costs."}],["DS_jjiywlyfpnme_0",{"n":"HO_LiMoNiTi_NPJCM_2020_bulk_water_train_test","d":"Training and testing configurations of bulk water from HO_LiMoNiTi_NPJCM_2020 used in the training of an ANN, whereby total energy is extrapolated by a Taylor expansion as a means of reducing computational costs."}],["DS_w3whk1nr1bld_0",{"n":"HO_LiMoNiTi_NPJCM_2020_bulk_water_validation","d":"Validation configurations of bulk water from HO_LiMoNiTi_NPJCM_2020 used in the training of an ANN, whereby total energy is extrapolated by a Taylor expansion as a means of reducing computational costs."}],["DS_or3nu4t64mvk_0",{"n":"HO_LiMoNiTi_NPJCM_2020_water_clusters","d":"Configurations of water clusters from HO_LiMoNiTi_NPJCM_2020 used in the training of an ANN, whereby total energy is extrapolated by a Taylor expansion as a means of reducing computational costs."}],["DS_o73heom14iew_0",{"n":"HO_PNAS_2019","d":"1590 configurations of H2O/water with total energy and forces calculated using a hybrid approach at DFT/revPBE0-D3 level of theory."}],["DS_fhvr3y1alb97_0",{"n":"HPt_NC_2022","d":"A training dataset of 90,000 configurations with interaction properties between H2 and Pt(111) surfaces."}],["DS_lvixye0ynk1o_0",{"n":"H_nature_2022","d":"Over 300,000 configurations in an expanded dataset of 19 hydrogen combustion reaction channels. Intrinsic reaction coordinate calculations (IRC) are combined with ab initio simulations (AIMD) and normal mode displacement (NM) calculations."}],["DS_qgi1kbtdzwmm_0",{"n":"HfO2_DPGEN_PRB_2021","d":"Approximately 28,500 configurations of hafnia (HfO2) used in the training of a DP model for the prediction of properties of various hafnia polymorphs, including transition barriers between different phases."}],["DS_u40eq96ge40x_0",{"n":"HfO2_NPJ_2020","d":"6000 configurations of liquid and amorphous HfO2 generated for use with an active learning ML model."}],["DS_r1sl5dye35kg_0",{"n":"Hydrogen-induced_insulating_state_SmNiO3","d":"A dataset of DFT-calculated energies created to investigate the effect of hydrogen doping on the crystal structure and the electronic state in SmNiO3.Configuration sets include sets for apically and side-bonded hydrogen atoms for 1-9 hydrogen atoms."}],["DS_ngsaypfb9rnj_0",{"n":"ISO17_NC_2017","d":"129 molecules of composition C7O2H10 from the QM9 dataset with 5000 conformational geometries apiece. Molecular dynamics data was simulated using the Fritz-Haber Institute ab initio simulation software."}],["DS_v4e6t79tt5xh_0",{"n":"In2Se3_2D_DPGEN","d":"Approximately 11,500 configurations of In2Se3, including monolayer (20-atom slab) and bulk (30-atom supercell) models."}],["DS_03ph25s4yepy_0",{"n":"InP_JPCA2020","d":"This data set was used to generate a multi-element linear SNAP potential for InP, as published in Cusentino, M. A. et. al, J. Chem. Phys. (2020). Intended to produce an interatomic potential for indium phosphide capable of capturing high-energy defects that result from radiation damage cascades."}],["DS_hdv6si8yu2mv_0",{"n":"JARVIS_2DMatPedia","d":"The JARVIS-2DMatPedia dataset is part of the joint automated repository for various integrated simulations (JARVIS) database. This subset contains configurations with 2D materials from the 2DMatPedia database, generated through two methods: a top-down exfoliation approach, using structures of bulk materials from the Materials Project database; and a bottom-up approach, replacing each element in a 2D material with another from the same group (according to column number). JARVIS is a set of tools and datasets built to meet current materials design challenges."}],["DS_z3s0qui5vg5c_0",{"n":"JARVIS_AGRA_CHO","d":"The JARVIS_AGRA_CHO dataset is part of the joint automated repository for various integrated simulations (JARVIS) DFT database. This dataset contains data from the CO2 reduction reaction (CO2RR) dataset from Chen et al., as used in the automated graph representation algorithm (AGRA) training dataset: a collection of DFT training data for training a graph representation method to extract the local chemical environment of metallic surface adsorption sites. JARVIS is a set of tools and datasets built to meet current materials design challenges."}],["DS_1ynp8v7d3er5_0",{"n":"JARVIS_AGRA_CO","d":"The JARVIS_AGRA_CO dataset is part of the joint automated repository for various integrated simulations (JARVIS) DFT database. This dataset contains data from the CO2 reduction reaction (CO2RR) dataset from Chen et al., as used in the automated graph representation algorithm (AGRA) training dataset: a collection of DFT training data for training a graph representation method to extract the local chemical environment of metallic surface adsorption sites. JARVIS is a set of tools and datasets built to meet current materials design challenges."}],["DS_bggn2p6qw8zv_0",{"n":"JARVIS_AGRA_COOH","d":"The JARVIS_AGRA_COOH dataset is part of the joint automated repository for various integrated simulations (JARVIS) DFT database. This dataset contains data from the CO2 reduction reaction (CO2RR) dataset from Chen et al., as used in the automated graph representation algorithm (AGRA) training dataset: a collection of DFT training data for training a graph representation method to extract the local chemical environment of metallic surface adsorption sites. JARVIS is a set of tools and datasets built to meet current materials design challenges."}],["DS_ikx8oaixaoz0_0",{"n":"JARVIS_AGRA_O","d":"The JARVIS_AGRA_O dataset is part of the joint automated repository for various integrated simulations (JARVIS) DFT database. This dataset contains data from the training set for the oxygen reduction reaction (ORR) dataset from Batchelor et al., as used in the automated graph representation algorithm (AGRA) training dataset: a collection of DFT training data for training a graph representation method to extract the local chemical environment of metallic surface adsorption sites.  Bulk calculations were performed with k-point = 8 x 8 x 4. Training adsorption energies were calculated on slabs, k-point = 4 x 4 x 1, while testing energies used k-point = 3 x 3 x 1. JARVIS is a set of tools and datasets built to meet current materials design challenges."}],["DS_osavzsrz5mgf_0",{"n":"JARVIS_AGRA_OH","d":"The JARVIS_AGRA_OH dataset is part of the joint automated repository for various integrated simulations (JARVIS) DFT database. This dataset contains data from the training set for the oxygen reduction reaction (ORR) dataset from Batchelor et al., as used in the automated graph representation algorithm (AGRA) training dataset: a collection of DFT training data for training a graph representation method to extract the local chemical environment of metallic surface adsorption sites.  Bulk calculations were performed with k-point = 8 x 8 x 4. Training adsorption energies were calculated on slabs, k-point = 4 x 4 x 1, while testing energies used k-point = 3 x 3 x 1. JARVIS is a set of tools and datasets built to meet current materials design challenges."}],["DS_720rbshv96l1_0",{"n":"JARVIS_ALIGNN_FF","d":"The JARVIS_ALIGNN_FF dataset is part of the joint automated repository for various integrated simulations (JARVIS) database. This dataset is a subset of the JARVIS DFT dataset, filtered to contain just the first, last, middle, maximum energy and minimum energy structures. Additionally, calculation run snapshots are filtered for uniqueness, and the dataset contains only perfect structures. DFT energies, stresses and forces in this dataset were used to train an atomisitic line graph neural network (ALIGNN)-based FF model. JARVIS is a set of tools and datasets built to meet current materials design challenges."}],["DS_8hgxhsfkcfa7_0",{"n":"JARVIS_C2DB","d":"The JARVIS-C2DB dataset is part of the joint automated repository for various integrated simulations (JARVIS) database. This subset contains configurations from the Computational 2D Database (C2DB), which contains a variety of properties for 2-dimensional materials across more than 30 differentcrystal structures. JARVIS is a set of tools and datasets built to meet current materials design challenges."}],["DS_np1noc8iip47_0",{"n":"JARVIS_CFID_3D_8_18_2022","d":"The JARVIS_CFID_3D_8_18_2022 dataset is part of the joint automated repository for various integrated simulations (JARVIS) DFT database. This subset contains configurations of 3D materials. JARVIS is a set of tools and datasets built to meet current materials design challenges."}],["DS_u8strp7hm0cy_0",{"n":"JARVIS_CFID_OQMD","d":"The JARVIS_CFID_OQMD dataset is part of the joint automated repository for various integrated simulations (JARVIS) database. This dataset contains configurations from the Open Quantum Materials Database (OQMD), created to hold information about the electronic structure and stability of organic materials for the purpose of aiding in materials discovery. Calculations were performed at the DFT level of theory, using the PAW-PBE functional implemented by VASP. This dataset also includes classical force-field inspired descriptors (CFID) for each configuration. JARVIS is a set of tools and collected datasets built to meet current materials design challenges."}],["DS_4ml1yrigmar0_0",{"n":"JARVIS_DFT_2D_3_12_2021","d":"The DFT-2D-3-12-2021 dataset is part of the joint automated repository for various integrated simulations (JARVIS) DFT database. This subset contains configurations of 2D materials. JARVIS is a set of tools and datasets built to meet current materials design challenges."}],["DS_t02h9eraswtv_0",{"n":"JARVIS_DFT_3D_12_12_2022","d":"The DFT_3D_12_12_2022 dataset is part of the joint automated repository for various integrated simulations (JARVIS) DFT database. This subset contains configurations of 3D materials. JARVIS is a set of tools and datasets built to meet current materials design challenges."}],["DS_soziqjohq6hm_0",{"n":"JARVIS_DFT_3D_8_18_2021","d":"The JARVIS_DFT_3D_8_18_2021 dataset is part of the joint automated repository for various integrated simulations (JARVIS) DFT database. This subset contains configurations of 3D materials. JARVIS is a set of tools and datasets built to meet current materials design challenges."}],["DS_7jji22dy5hix_0",{"n":"JARVIS_EPC_2D","d":"The JARVIS_EPC_2D dataset is part of the joint automated repository for various integrated simulations (JARVIS) DFT database. This subset contains configurations sourced from the JARVIS-DFT-2D dataset, rerelaxed with Quantum ESPRESSO. JARVIS is a set of tools and datasets built to meet current materials design challenges."}],["DS_mvuwxu67yrdy_0",{"n":"JARVIS_HOPV","d":"The JARVIS_HOPV dataset is part of the joint automated repository for various integrated simulations (JARVIS) database. This dataset contains configurations from the Harvard organic photovoltaic (HOPV) dataset, collated experimental data from the literature. Quantum-chemical calculations are performed using five functionals: NP86, PBE0, B3LYP, M06-2X and basis sets. In addition to HOMO, LUMO and gap energies, calculations include open circuit potentials (Voc), short circuit current density (Jsc) and percent conversion efficiency (PCE). JARVIS is a set of tools and collected datasets built to meet current materials design challenges."}],["DS_9yr94hhj1k33_0",{"n":"JARVIS_MEGNet","d":"The JARVIS-MEGNet dataset is part of the joint automated repository for various integrated simulations (JARVIS) database. This subset contains configurations with 3D materials properties from the 2018 version of Materials Project, as used in the training of the MEGNet ML model. JARVIS is a set of tools and datasets built to meet current materials design challenges."}],["DS_5ar73nonq6l1_0",{"n":"JARVIS_MEGNet2","d":"The JARVIS-MEGNet2 dataset is part of the joint automated repository for various integrated simulations (JARVIS) database. This subset contains 133K materials with formation energy from the Materials Project, as used in the training of the MEGNet ML model. JARVIS is a set of tools and datasets built to meet current materials design challenges."}],["DS_gpsibs9f47k4_0",{"n":"JARVIS_Materials_Project_2020","d":"The JARVIS_Materials_Project_2020 dataset is part of the joint automated repository for various integrated simulations (JARVIS) DFT database. This subset contains 127,000 configurations of 3D materials from the Materials Project database. JARVIS is a set of tools and datasets built to meet current materials design challenges."}],["DS_1s8172fnm2ct_0",{"n":"JARVIS_Materials_Project_84K","d":"The JARVIS_Materials_Project_84K dataset is part of the joint automated repository for various integrated simulations (JARVIS) DFT database. This subset contains 84,000 configurations of 3D materials from the Materials Project database. JARVIS is a set of tools and datasets built to meet current materials design challenges."}],["DS_hrt0twm503tr_0",{"n":"JARVIS_OMDB","d":"The JARVIS_OMDB dataset is part of the joint automated repository for various integrated simulations (JARVIS) database. This dataset contains configurations from the Organic Materials Database (OMDB): a dataset of 12,500 crystal materials for the purpose of training models for the prediction of properties for complex and lattice-periodic organic crystals with large numbers of atoms per unit cell. Dataset covers 69 space groups, 65 elements; averages 82 atoms per unit cell. This dataset also includes classical force-field inspired descriptors (CFID) for each configuration. JARVIS is a set of tools and collected datasets built to meet current materials design challenges."}],["DS_untxw8nljf92_0",{"n":"JARVIS_OQMD_no_CFID","d":"The JARVIS_OQMD_no_CFID dataset is part of the joint automated repository for various integrated simulations (JARVIS) database. This dataset contains configurations from the Open Quantum Materials Database (OQMD), created to hold information about the electronic structure and stability of organic materials for the purpose of aiding in materials discovery. Calculations were performed at the DFT level of theory, using the PAW-PBE functional implemented by VASP. JARVIS is a set of tools and collected datasets built to meet current materials design challenges."}],["DS_armtbsouma25_0",{"n":"JARVIS_Open_Catalyst_100K","d":"The JARVIS_Open_Catalyst_100K dataset is part of the joint automated repository for various integrated simulations (JARVIS) DFT database. This subset contains configurations from the 100K training, rest validation and test dataset from the Open Catalyst Project (OCP). JARVIS is a set of tools and datasets built to meet current materials design challenges."}],["DS_6dcul0b17524_0",{"n":"JARVIS_Open_Catalyst_10K","d":"The JARVIS_Open_Catalyst_10K dataset is part of the joint automated repository for various integrated simulations (JARVIS) DFT database. This subset contains configurations from the 10K training, rest validation and test dataset from the Open Catalyst Project (OCP). JARVIS is a set of tools and datasets built to meet current materials design challenges."}],["DS_3u17g7ukggi5_0",{"n":"JARVIS_Open_Catalyst_All","d":"The JARVIS_Open_Catalyst_All dataset is part of the joint automated repository for various integrated simulations (JARVIS) DFT database. This subset contains configurations from the Open Catalyst Project (OCP) 460328 training, rest validation and test dataset. JARVIS is a set of tools and datasets built to meet current materials design challenges."}],["DS_dbgckv1il6v7_0",{"n":"JARVIS_Polymer-Genome","d":"The JARVIS-Polymer-Genome dataset is part of the joint automated repository for various integrated simulations (JARVIS) database. This dataset contains configurations from the Polymer Genome dataset, as created for the linked publication (Huan, T., Mannodi-Kanakkithodi, A., Kim, C. et al.). Structures were curated from existing sources and the original authors' works, removing redundant, identical structures before calculations, and removing redundant datapoints after calculations were performed. Band gap energies were calculated using two different DFT functionals: rPW86 and HSE06; atomization energy was calculated using rPW86. JARVIS is a set of tools and collected datasets built to meet current materials design challenges."}],["DS_e471qdt7c6db_0",{"n":"JARVIS_QE_TB","d":"The QE-TB dataset is part of the joint automated repository for various integrated simulations (JARVIS) DFT database. This subset contains configurations generated in Quantum ESPRESSO. JARVIS is a set of tools and datasets built to meet current materials design challenges."}],["DS_tat5i46x3hkr_0",{"n":"JARVIS_QM9-DGL","d":"The JARVIS-QM9-DGL dataset is part of the joint automated repository for various integrated simulations (JARVIS) database. This dataset contains configurations from the QM9 dataset, originally created as part of the datasets at quantum-machine.org, as implemented with the Deep Graph Library (DGL) Python package. Units for r2 (electronic spatial extent) are a0^2; for alpha (isotropic polarizability), a0^3; for mu (dipole moment), D; for Cv (heat capacity), cal/mol K. Units for all other properties are eV. JARVIS is a set of tools and collected datasets built to meet current materials design challenges."}],["DS_jz1q9juw7ycj_0",{"n":"JARVIS_QM9_STD_JCTC","d":"The JARVIS_QM9_STD_JCTC dataset is part of the joint automated repository for various integrated simulations (JARVIS) database. This dataset contains configurations from the QM9 dataset, originally created as part of the datasets at quantum-machine.org. Units for r2 (electronic spatial extent) are a\u0000^2; for alpha (isotropic polarizability), a\u0000^3; for mu (dipole moment), D; for Cv (heat capacity), cal/mol K. Units for all other properties are eV. JARVIS is a set of tools and collected datasets built to meet current materials design challenges.For the first iteration of DFT calculations, Gaussian 09\u2019s default electronic and geometry thresholds have been used for all molecules. For those molecules which failed to reach SCF convergence ultrafine grids have been invoked within a second iteration for evaluating the XC energy contributions. Within a third iteration on the remaining unconverged molecules, we identified those which had relaxed to saddle points, and further tightened the SCF criteria using the keyword scf(maxcycle=200, verytight). All those molecules which still featured imaginary frequencies entered the fourth iteration using keywords, opt(calcfc, maxstep=5, maxcycles=1000). calcfc constructs a Hessian in the first step of the geometry relaxation for eigenvector following. Within the fifth and final iteration, all molecules which still failed to reach convergence, have subsequently been converged using opt(calcall, maxstep=1, maxcycles=1000)"}],["DS_221svb9fxfk7_0",{"n":"JARVIS_QMOF","d":"The JARVIS_QMOF dataset is part of the joint automated repository for various integrated simulations (JARVIS) database. This dataset contains configurations from the Quantum Metal-Organic Frameworks (QMOF) dataset, comprising quantum-chemical properties for >14,000 experimentally synthesized MOFs. QMOF contains \"DFT-ready\" data: filtered to remove omitted, overlapping, unbonded or deleted atoms, along with other kinds of problematic structures commented on in the literature. Data were generated via high-throughput DFT workflow, at the PBE-D3(BJ) level of theory using VASP software. JARVIS is a set of tools and collected datasets built to meet current materials design challenges."}],["DS_1nbddfnjxbjc_0",{"n":"JARVIS_SNUMAT","d":"The JARVIS_SNUMAT dataset is part of the joint automated repository for various integrated simulations (JARVIS) database. This dataset contains band gap data for >10,000 materials, computed using a hybrid functional and considering the stable magnetic ordering. Structure relaxation and band edges are obtained using the PBE XC functional; band gap energy is subsequently obtained using the HSE06 hybrid functional. Optical and fundamental band gap energies are included. Some gap energies are recalculated by including spin-orbit coupling. These are noted in the band gap metadata as \"SOC=true\". JARVIS is a set of tools and collected datasets built to meet current materials design challenges."}],["DS_ffsgl0wufoft_0",{"n":"JARVIS_TinNet_N","d":"The JARVIS_TinNet dataset is part of the joint automated repository for various integrated simulations (JARVIS) database. This dataset contains configurations from the TinNet-N dataset: a collection assembled to train a machine learning model for the purposes of assisting catalyst design by predicting chemical reactivity of transition-metal surfaces. The adsorption systems contained in this dataset consist of {100}-terminated Pt-based bimetallic surfaces doped with a third element. JARVIS is a set of tools and collected datasets built to meet current materials design challenges."}],["DS_ok9dbnj53zih_0",{"n":"JARVIS_TinNet_O","d":"The JARVIS_TinNet dataset is part of the joint automated repository for various integrated simulations (JARVIS) database. This dataset contains configurations from the TinNet-O dataset: a collection assembled to train a machine learning model for the purposes of assisting catalyst design by predicting chemical reactivity of transition-metal surfaces. The adsorption systems contained in this dataset consist of {111}-terminated metal surfaces. JARVIS is a set of tools and collected datasets built to meet current materials design challenges."}],["DS_mk2e641eop9w_0",{"n":"JARVIS_TinNet_OH","d":"The JARVIS_TinNet dataset is part of the joint automated repository for various integrated simulations (JARVIS) database. This dataset contains configurations from the TinNet-OH dataset: a collection assembled to train a machine learning model for the purposes of assisting catalyst design by predicting chemical reactivity of transition-metal surfaces. The adsorption systems contained in this dataset consist of {111}-terminated metal surfaces. JARVIS is a set of tools and collected datasets built to meet current materials design challenges."}],["DS_g9sra4y8efji_0",{"n":"JARVIS_mlearn","d":"The JARVIS_mlearn dataset is part of the joint automated repository for various integrated simulations (JARVIS) database. This dataset contains configurations from the Organic Materials Database (OMDB): a dataset of 12,500 crystal materials for the purpose of training models for the prediction of properties for complex and lattice-periodic organic crystals with large numbers of atoms per unit cell. Dataset covers 69 space groups, 65 elements; averages 82 atoms per unit cell. This dataset also includes classical force-field inspired descriptors (CFID) for each configuration. JARVIS is a set of tools and collected datasets built to meet current materials design challenges."}],["DS_4lev0d7cs1yl_0",{"n":"LiGePS_SSE_PBE","d":"Approximately 6,500 configurations of Li10GeP2S12, based on crystal structures from the Materials Project database, material ID mp-696129. One of two LiGePS datasets from this source. The other uses the PBEsol functional, rather than the PBE functional."}],["DS_dhtzh0y2108p_0",{"n":"LiGePS_SSE_PBEsol","d":"Approximately 2,800 configurations of Li10GeP2S12, based on crystal structures from the Materials Project database, material ID mp-696129. One of two LiGePS datasets from this source. The other uses the PBE functional, rather than the PBEsol functional."}],["DS_9p3sip4yhiju_0",{"n":"LiSiPS_SSE_PBE","d":"Approximately 9,100 configurations of Li10SiP2S12, based on crystal structures from the Materials Project database, material ID mp-696129. One of two LiSiPS datasets from this source. The other uses the PBEsol functional, rather than the PBE functional."}],["DS_th9w66lyxspv_0",{"n":"LiSiPS_SSE_PBEsol","d":"Approximately 2,300 configurations of Li10SiP2S12, based on crystal structures from the Materials Project database, material ID mp-696129. One of two LiSiPS datasets from this source. The other uses the PBE functional, rather than the PBEsol functional."}],["DS_0pv4lpmx2ov3_0",{"n":"LiTiO_Science_2020","d":"This dataset contains configurations of lithium titanate from the publication Kinetic Pathways of ionic transport in fast-charging lithium titanate. In order to understand the origin of various EELS (electron energy-loss spectroscopy) spectra features, EELS spectra were simulated using the Vienna Ab initio Simulation (VASP) package. For a specific Li in a given configuration, this is done by calculating the DOS and integrated DOS considering a Li core-hole on the position of the specific Li and calculating the EELS based on the DOS. The minimum energy paths (MEP) and migration energy of Li were calculated in various compositions, including Li4Ti5O12 with an additional Li carrier, Li5Ti5O12 with an additional Li carrier, and Li7Ti5O12 with a Li vacancy carrier."}],["DS_eqt5dbhsmm68_0",{"n":"MD22_AT_AT","d":"Dataset containing MD trajectories of AT-AT DNA base pairs from the MD22 benchmark set. {DESC}"}],["DS_rx1ei5q0x9gy_0",{"n":"MD22_AT_AT_CG_CG","d":"Dataset containing MD trajectories of AT-AT-CG-CG DNA base pairs from the MD22 benchmark set. MD22 represents a collection of datasets in a benchmark that can be considered an updated version of the MD17 benchmark datasets, including more challenges with respect to system size, flexibility and degree of non-locality. The datasets in MD22 include MD trajectories of the protein Ac-Ala3-NHMe; the lipid DHA (docosahexaenoic acid); the carbohydrate stachyose; nucleic acids AT-AT and AT-AT-CG-CG; and the buckyball catcher and double-walled nanotube supramolecules. Each of these is included here in a separate dataset, as represented on sgdml.org. Calculations were performed using FHI-aims and i-Pi software at the DFT-PBE+MBD level of theory. Trajectories were sampled at temperatures between 400-500 K at 1 fs resolution."}],["DS_l6awe3jnivnm_0",{"n":"MD22_Ac_Ala3_NHMe","d":"Dataset containing MD trajectories of the 42-atom tetrapeptide Ac-Ala3-NHMe from the MD22 benchmark set. MD22 represents a collection of datasets in a benchmark that can be considered an updated version of the MD17 benchmark datasets, including more challenges with respect to system size, flexibility and degree of non-locality. The datasets in MD22 include MD trajectories of the protein Ac-Ala3-NHMe; the lipid DHA (docosahexaenoic acid); the carbohydrate stachyose; nucleic acids AT-AT and AT-AT-CG-CG; and the buckyball catcher and double-walled nanotube supramolecules. Each of these is included here in a separate dataset, as represented on sgdml.org. Calculations were performed using FHI-aims and i-Pi software at the DFT-PBE+MBD level of theory. Trajectories were sampled at temperatures between 400-500 K at 1 fs resolution."}],["DS_xwmthngerf71_0",{"n":"MD22_DHA","d":"Dataset containing MD trajectories of DHA (docosahexaenoic acid) from the MD22 benchmark set. MD22 represents a collection of datasets in a benchmark that can be considered an updated version of the MD17 benchmark datasets, including more challenges with respect to system size, flexibility and degree of non-locality. The datasets in MD22 include MD trajectories of the protein Ac-Ala3-NHMe; the lipid DHA (docosahexaenoic acid); the carbohydrate stachyose; nucleic acids AT-AT and AT-AT-CG-CG; and the buckyball catcher and double-walled nanotube supramolecules. Each of these is included here in a separate dataset, as represented on sgdml.org. Calculations were performed using FHI-aims and i-Pi software at the DFT-PBE+MBD level of theory. Trajectories were sampled at temperatures between 400-500 K at 1 fs resolution."}],["DS_7g8q68fszjvs_0",{"n":"MD22_buckyball_catcher","d":"Dataset containing MD trajectories of the buckyball-catcher supramolecule from the MD22 benchmark set. MD22 represents a collection of datasets in a benchmark that can be considered an updated version of the MD17 benchmark datasets, including more challenges with respect to system size, flexibility and degree of non-locality. The datasets in MD22 include MD trajectories of the protein Ac-Ala3-NHMe; the lipid DHA (docosahexaenoic acid); the carbohydrate stachyose; nucleic acids AT-AT and AT-AT-CG-CG; and the buckyball catcher and double-walled nanotube supramolecules. Each of these is included here in a separate dataset, as represented on sgdml.org. Calculations were performed using FHI-aims and i-Pi software at the DFT-PBE+MBD level of theory. Trajectories were sampled at temperatures between 400-500 K at 1 fs resolution."}],["DS_d92bnafzwynr_0",{"n":"MD22_double_walled_nanotube","d":"Dataset containing MD trajectories of the double-walled nanotube supramolecule from the MD22 benchmark set. MD22 represents a collection of datasets in a benchmark that can be considered an updated version of the MD17 benchmark datasets, including more challenges with respect to system size, flexibility and degree of non-locality. The datasets in MD22 include MD trajectories of the protein Ac-Ala3-NHMe; the lipid DHA (docosahexaenoic acid); the carbohydrate stachyose; nucleic acids AT-AT and AT-AT-CG-CG; and the buckyball catcher and double-walled nanotube supramolecules. Each of these is included here in a separate dataset, as represented on sgdml.org. Calculations were performed using FHI-aims and i-Pi software at the DFT-PBE+MBD level of theory. Trajectories were sampled at temperatures between 400-500 K at 1 fs resolution."}],["DS_87ojug1k96ef_0",{"n":"MD22_stachyose","d":"Dataset containing MD trajectories of the tetrasaccharide stachyose from the MD22 benchmark set. MD22 represents a collection of datasets in a benchmark that can be considered an updated version of the MD17 benchmark datasets, including more challenges with respect to system size, flexibility and degree of non-locality. The datasets in MD22 include MD trajectories of the protein Ac-Ala3-NHMe; the lipid DHA (docosahexaenoic acid); the carbohydrate stachyose; nucleic acids AT-AT and AT-AT-CG-CG; and the buckyball catcher and double-walled nanotube supramolecules. Each of these is included here in a separate dataset, as represented on sgdml.org. Calculations were performed using FHI-aims and i-Pi software at the DFT-PBE+MBD level of theory. Trajectories were sampled at temperatures between 400-500 K at 1 fs resolution."}],["DS_rsao7xrpu9ig_0",{"n":"MISPR","d":"Example dataset for MISPR (Materials Informatics for Structure-Property Relationships) materials science simulation software, with DFT-calculated configuration properties for three different MISPR workflows: nuclear magnetic resonance (NMR) chemical shifts, electrostatic partial charges (ESP) and bond dissociation energies (BDE)."}],["DS_326i4urabisb_0",{"n":"MTPu_2023","d":"A comprehensive database generated using density functional theory simulations, encompassing a wide range of crystal structures, point defects, extended defects, and disordered structure."}],["DS_5drebe4tktiu_0",{"n":"Matbench_mp_e_form","d":"Matbench v0.1 test dataset for predicting DFT formation energy from structure. Adapted from Materials Project database. Entries having formation energy more than 2.5eV and those containing noble gases are removed. Retrieved April 2, 2019. For benchmarking w/ nested cross validation, the order of the dataset must be identical to the retrieved data; refer to the Automatminer/Matbench publication for more details.Matbench is an automated leaderboard for benchmarking state of the art ML algorithms predicting a diverse range of solid materials' properties. It is hosted and maintained by the Materials Project."}],["DS_6lq2f26dluql_0",{"n":"Matbench_mp_gap","d":"The Matbench_mp_gap dataset is a Matbench v0.1 test dataset for predicting DFT PBE band gap from structure, adapted from the Materials Project database. Entries having a formation energy (or energy above the convex hull) greater than 150meV and those containing noble gases have been removed. Retrieved April 2, 2019. Refer to the Automatminer/Matbench publication for more details. This dataset contains band gap as calculated by PBE DFT from the Materials Project, in eV. Matbench is an automated leaderboard for benchmarking state of the art ML algorithms predicting a diverse range of solid materials' properties. It is hosted and maintained by the Materials Project."}],["DS_sinisy47w8ff_0",{"n":"Matbench_perovskites","d":"The Matbench_perovskites dataset is a Matbench v0.1 test dataset for predicting formation energy from crystal structure. Adapted from an original dataset generated by Castelli et al. Refer to the Automatminer/Matbench publication for more details. This dataset contains the energy of formation of the entire 5-atom perovskite cell in eV as calculated by RPBE GGA-DFT. Note the reference state for oxygen was computed from oxygen's chemical potential in water vapor, not as oxygen molecules, to reflect the application for which these perovskites were studied. Matbench is an automated leaderboard for benchmarking state of the art ML algorithms predicting a diverse range of solid materials' properties. It is hosted and maintained by the Materials Project."}],["DS_pv1f3dlo5dsc_0",{"n":"Materials_Project","d":"Configurations from the Materials Project database: an online resource with the goal of computing properties of all inorganic materials."}],["DS_caktb6z8yiy7_0",{"n":"Mg_edmonds_2022","d":"16748 configurations of magnesium with gathered energy, stress and forces at the DFT level of theory."}],["DS_0shp3qrqk9k9_0",{"n":"MoNbTaVW_PRB2021","d":"This dataset was originally designed to fit a GAP model for the Mo-Nb-Ta-V-W quinary system that was used to study segregation and defects in the body-centered-cubic refractory high-entropy alloy MoNbTaVW."}],["DS_5aeg7va6k305_0",{"n":"Mo_PRM2019","d":"This dataset was designed to enable machine learning of Mo elastic, thermal, and defect properties, as well as surface energetics, melting, and the structure of the liquid phase. The dataset was constructed by starting with the dataset from J. Byggm\u00e4star et al., Phys. Rev. B 100, 144105 (2019), then rescaling all of the configurations to the correct lattice spacing and adding in gamma surface configurations."}],["DS_oqqwzogut1on_0",{"n":"NDSC_TUT_2022","d":"500 configurations of Mg2 for MD prediction using a model fitted on Al, W, Mg and Si."}],["DS_0j2smy6relq0_0",{"n":"NENCI-2021","d":"NENCI-2021 is a database of approximately 8000 benchmark Non-Equilibirum Non-Covalent Interaction (NENCI) energies performed on molecular dimers;intermolecular complexes of biological and chemical relevance with a particular emphasis on close intermolecular contacts. Based on dimersfrom the S101 database."}],["DS_yg0yzoiwoux8_0",{"n":"NEP_PRB_2021","d":"Approximately 7,000 distinct configurations of 2D-silicene, silicon, and PbTe. Silicon data used from http://dx.doi.org/10.1103/PhysRevX.8.041048. Dataset includes predicted force, potential energy and virial values."}],["DS_mc8p14cpn2ea_0",{"n":"NEP_qHPF_test","d":"The test set of a train and test set pair.The combined datasets comprise approximately 275 configurations of monolayer quasi-hexagonal-phase fullerene (qHPF) membrane used to train and test an NEP model."}],["DS_umliq1qh50gy_0",{"n":"NEP_qHPF_train","d":"The train set of a train and test set pair.The combined datasets comprise approximately 275 configurations of monolayer quasi-hexagonal-phase fullerene (qHPF) membrane used to train and test an NEP model."}],["DS_of1clsgf4nab_0",{"n":"NMD-18","d":"3,000 Al-Ga-In sesquioxides with energies and band gaps. Relaxed and Vegard's Law geometries with formation energy and band gaps at DFT-PBE level of theory of (Alx-Gay-Inz)2O3 oxides, x+y+z=1. Contains all structures from the NOMAD 2018 Kaggle challenge training and leaderboard data. The formation energy and bandgap energy were computed by using the PBE exchange-correlation DFT functional with the all-electron electronic structure code FHI-aims with tight setting."}],["DS_nb8hcpibz1dt_0",{"n":"NNIP_FeH_PRM_2021","d":"Approximately 20,000 configurations from a dataset of alpha-iron and hydrogen. Properties include forces and potential energy, calculated using VASP at the DFT level using the GGA-PBE functional."}],["DS_3qumwe2j8lib_0",{"n":"NNP-Ga2O3","d":"9,200 configurations of beta-Ga2O3, including two configuration sets. One contains DFT data for 8400 configurations simulated between temperatures of 50K - 600K. The second contains configurations with 0K simulation temperature."}],["DS_0gyf3srv7xh7_0",{"n":"NVNMD_GeTe","d":"Approximately 5,000 configurations of GeTe used in training of a non-von Neumann multiplication-less DNN model."}],["DS_p74559sdjy1q_0",{"n":"N_O_F_columns_non-bonded_vdW_potential_JCP2023","d":"This dataset contains structures of materials from the N (15th), O (16th) and F (16th) columns of the periodic table used for generating a 2-body non-bonded vdW potential."}],["DS_tbpx3pawtlgt_0",{"n":"Nb_PRM2019","d":"This dataset was designed to enable machine-learning of Nb elastic, thermal, and defect properties, as well as surface energetics, melting, and the structure of the liquid phase. The dataset was constructed by starting with the dataset from J. Byggm\u00e4star et al., Phys. Rev. B 100, 144105 (2019), then rescaling all of the configurations to the correct lattice spacing and adding in gamma surface configurations."}],["DS_4pbhjtu62o2d_0",{"n":"NequIP_NC_2022","d":"Approximately 57,000 configurations from the evaluation datasets for NequIP graph neural network model for interatomic potentials. Trajectories have been taken from LIPS, LIPO glass melt-quench simulation, and formate decomposition on Cu datasets. "}],["DS_5tegg4uvaixz_0",{"n":"NiCoCr_NC2020","d":"The face-centered cubic medium-entropy alloy NiCoCr has received considerable attention for its good mechanical properties, uncertain stacking fault energy, etc, some of which have been attributed to chemical short-range order (SRO). Here, we examine the yield strength and misfit volumes of NiCoCr to determine whether SRO has measurably influenced mechanical properties. Polycrystalline strengths show no systematic trend with different processing conditions. Measured misfit volumes in NiCoCr are consistent with those in random binaries. Yield strength prediction of a random NiCoCr alloy matches well with experiments. Finally, we show that standard spin-polarized density functional theory (DFT) calculations of misfit volumes are not accurate for NiCoCr. This implies that DFT may be inaccurate for other subtle structural quantities such as atom-atom bond distance so that caution is required in drawing conclusions about NiCoCr based on DFT. These findings all lead to the conclusion that, under typical processing conditions, SRO in NiCoCr is either negligible or has no systematic measurable effect on strength."}],["DS_3h39eqiv9urv_0",{"n":"OC22-IS2RE-Train","d":"Training configurations for the initial structure to relaxed total energy (IS2RE) task of OC22. Open Catalyst 2022 (OC22) is a database of training trajectories for predicting catalytic reactions on oxide surfaces meant to complement OC20, which did not contain oxide surfaces."}],["DS_4eb78xs9suoo_0",{"n":"OC22-IS2RE-Validation-in-domain","d":"In-domain validation configurations for the initial structure to relaxed total energy (IS2RE) task of OC22. Open Catalyst 2022 (OC22) is a database of training trajectories for predicting catalytic reactions on oxide surfaces meant to complement OC20, which did not contain oxide surfaces."}],["DS_ikbqbyd3dw25_0",{"n":"OC22-IS2RE-Validation-out-of-domain","d":"Out-of-domain validation configurations for the initial structure to relaxed total energy (IS2RE) task of OC22. Open Catalyst 2022 (OC22) is a database of training trajectories for predicting catalytic reactions on oxide surfaces meant to complement OC20, which did not contain oxide surfaces."}],["DS_jgaid7espcoc_0",{"n":"OC22-S2EF-Train","d":"Training configurations for the structure to total energy and forces task (S2EF) of OC22. Open Catalyst 2022 (OC22) is a database of training trajectories for predicting catalytic reactions on oxide surfaces meant to complement OC20, which did not contain oxide surfaces."}],["DS_f25kywgtdhks_0",{"n":"OC22-S2EF-Validation-in-domain","d":"In-domain validation configurations for the structure to total energy and forces (S2EF) task of OC22. Open Catalyst 2022 (OC22) is a database of training trajectories for predicting catalytic reactions on oxide surfaces meant to complement OC20, which did not contain oxide surfaces."}],["DS_y8u933mhadjf_0",{"n":"OC22-S2EF-Validation-out-of-domain","d":"Out-of-domain validation configurations for the structure to total energy and forces (S2EF) task of OC22. Open Catalyst 2022 (OC22) is a database of training trajectories for predicting catalytic reactions on oxide surfaces meant to complement OC20, which did not contain oxide surfaces."}],["DS_5obi9nxcgmof_0",{"n":"OrbNet_Denali","d":"All DFT single-point calculations for the OrbNet Denali training set were carried out in Entos Qcore version 0.8.17 at the \u03c9B97X-D3/def2-TZVP level of theory using in-core density fitting with the neese=4 DFT integration grid."}],["DS_cm3cr5kgqomw_0",{"n":"Paramagnetic_lanthanide_compounds","d":"This dataset is composed of fully-deuterated Gd(III) analogue d-[GdL] in a variety of solvent materials, including MeOH, D2O and d6-DMSO."}],["DS_l3xhsrlgu8tq_0",{"n":"PtNi_alloy_NPJ2022","d":"DFT dataset consisting of 6828 resampled Pt-Ni alloys used for training an NNP. The energy and forces of each structure in the resampled database are calculated using DFT. All reference DFT calculations for the training set of 6828 Pt-Ni alloy structures have been performed using the Vienna Ab initio Simulation Package (VASP) with the spin-polarized revised Perdew-Burke-Ernzerhof (rPBE) exchange-correlation functional."}],["DS_2qq05tdbyovn_0",{"n":"QM-22","d":"Includes CHON molecules of 4-15 atoms, developed in counterpoint to the MD17 dataset, run at higher total energies (above 500 K) and with a broader configuration space."}],["DS_3a5xucj4yqa8_0",{"n":"QM7b_AlphaML","d":"Energy, computed with LR-CCSD, hybrid DFT (B3LYP & SCAN0) for 7211 molecules in QM7b and 52 molecules in AlphaML showcase database."}],["DS_d5ug96vla5xy_0",{"n":"QM9x","d":"Dataset containing DFT calculations of energy and forces for all configurations in the QM9 dataset, recalculated with the \u03c9B97X functional and 6-31G(d) basis set. Recalculating the energy and forces causes a slight shift of the potential energy surface, which results in forces acting on most configurations in the dataset. The data was generated by running Nudged Elastic Band (NEB) calculations with DFT on 10k reactions while saving intermediate calculations. QM9x is used as a benchmarking and comparison dataset for the dataset Transition1x."}],["DS_02cqe6a0bobu_0",{"n":"QM_hamiltonian_nature_2019","d":"~100,000 configurations of water, ethanol, malondialdehyde and uracil gathered at the PBE/def2-SVP level of theory using ORCA."}],["DS_s818rozdb6x6_0",{"n":"REANN_CO2_Ni100","d":"Approximately 9,850 configurations of CO2 with a movable Ni(100) surface."}],["DS_dixgsms1jm98_0",{"n":"SAIT_semiconductors_ACS_2023_HfO_out-of-domain","d":"Out-of-domain configurations from the SAIT_semiconductors_ACS_2023_HfO dataset. This dataset contains HfO configurations from the SAIT semiconductors datasets. SAIT semiconductors datasets comprise two rich datasets for the important semiconductor thin film materials silicon nitride (SiN) and hafnium oxide (HfO), gathered for the development of MLFFs. DFT simulations were conducted under various conditions that include differing initial structures, stoichiometry, temperature, strain, and defects."}],["DS_ekrypue10aay_0",{"n":"SAIT_semiconductors_ACS_2023_HfO_raw","d":"Structures from the SAIT_semiconductors_ACS_2023_HfO dataset, separated into crystal, out-of-domain, and random (generated by randomly distributing 32 Hf and 64 O atoms within the unit cells of the HfO2 crystals) configuration sets. This dataset contains HfO configurations from the SAIT semiconductors datasets. SAIT semiconductors datasets comprise two rich datasets for the important semiconductor thin film materials silicon nitride (SiN) and hafnium oxide (HfO), gathered for the development of MLFFs. DFT simulations were conducted under various conditions that include differing initial structures, stoichiometry, temperature, strain, and defects."}],["DS_dxzkbbocjb0y_0",{"n":"SAIT_semiconductors_ACS_2023_HfO_test","d":"Test configurations from the SAIT_semiconductors_ACS_2023_HfO dataset. This dataset contains HfO configurations from the SAIT semiconductors datasets. SAIT semiconductors datasets comprise two rich datasets for the important semiconductor thin film materials silicon nitride (SiN) and hafnium oxide (HfO), gathered for the development of MLFFs. DFT simulations were conducted under various conditions that include differing initial structures, stoichiometry, temperature, strain, and defects."}],["DS_ppk44zeithjj_0",{"n":"SAIT_semiconductors_ACS_2023_HfO_train","d":"Training configurations from the SAIT_semiconductors_ACS_2023_HfO dataset. This dataset contains HfO configurations from the SAIT semiconductors datasets. SAIT semiconductors datasets comprise two rich datasets for the important semiconductor thin film materials silicon nitride (SiN) and hafnium oxide (HfO), gathered for the development of MLFFs. DFT simulations were conducted under various conditions that include differing initial structures, stoichiometry, temperature, strain, and defects."}],["DS_89w2iq8fw7qi_0",{"n":"SAIT_semiconductors_ACS_2023_HfO_validation","d":"Validation configurations from the SAIT_semiconductors_ACS_2023_HfO dataset. This dataset contains HfO configurations from the SAIT semiconductors datasets. SAIT semiconductors datasets comprise two rich datasets for the important semiconductor thin film materials silicon nitride (SiN) and hafnium oxide (HfO), gathered for the development of MLFFs. DFT simulations were conducted under various conditions that include differing initial structures, stoichiometry, temperature, strain, and defects."}],["DS_mo7yelvudt4k_0",{"n":"SAIT_semiconductors_ACS_2023_SiN_out-of-domain","d":"Out-of-domain configurations from the SAIT_semiconductors_ACS_2023_SiN dataset. This dataset contains SiN, Si and N configurations from the SAIT semiconductors datasets. SAIT semiconductors datasets comprise two rich datasets for the important semiconductor thin film materials silicon nitride (SiN) and hafnium oxide (HfO), gathered for the development of MLFFs. DFT simulations were conducted under various conditions that include differing initial structures, stoichiometry, temperature, strain, and defects."}],["DS_piuigd7monq9_0",{"n":"SAIT_semiconductors_ACS_2023_SiN_raw","d":"Structures from the SAIT_semiconductors_ACS_2023_SiN dataset, separated into N-only, Si-only, SiN, and out-of-domain melt, quench and relax configuration sets. This dataset contains SiN, Si and N configurations from the SAIT semiconductors datasets. SAIT semiconductors datasets comprise two rich datasets for the important semiconductor thin film materials silicon nitride (SiN) and hafnium oxide (HfO), gathered for the development of MLFFs. DFT simulations were conducted under various conditions that include differing initial structures, stoichiometry, temperature, strain, and defects."}],["DS_orfjpdomkekx_0",{"n":"SAIT_semiconductors_ACS_2023_SiN_test","d":"Test configurations from the SAIT_semiconductors_ACS_2023_SiN dataset. This dataset contains SiN, Si and N configurations from the SAIT semiconductors datasets. SAIT semiconductors datasets comprise two rich datasets for the important semiconductor thin film materials silicon nitride (SiN) and hafnium oxide (HfO), gathered for the development of MLFFs. DFT simulations were conducted under various conditions that include differing initial structures, stoichiometry, temperature, strain, and defects."}],["DS_5yfdgzb5zhgm_0",{"n":"SAIT_semiconductors_ACS_2023_SiN_train","d":"Training configurations from the SAIT_semiconductors_ACS_2023_SiN dataset. This dataset contains SiN, Si and N configurations from the SAIT semiconductors datasets. SAIT semiconductors datasets comprise two rich datasets for the important semiconductor thin film materials silicon nitride (SiN) and hafnium oxide (HfO), gathered for the development of MLFFs. DFT simulations were conducted under various conditions that include differing initial structures, stoichiometry, temperature, strain, and defects."}],["DS_5piytwom0j25_0",{"n":"SAIT_semiconductors_ACS_2023_SiN_validation","d":"Validation configurations from the SAIT_semiconductors_ACS_2023_SiN dataset. This dataset contains SiN, Si and N configurations from the SAIT semiconductors datasets. SAIT semiconductors datasets comprise two rich datasets for the important semiconductor thin film materials silicon nitride (SiN) and hafnium oxide (HfO), gathered for the development of MLFFs. DFT simulations were conducted under various conditions that include differing initial structures, stoichiometry, temperature, strain, and defects."}],["DS_w2ngtxl0ep5d_0",{"n":"SIMPLE_NN_SiO2","d":"10,000 configurations of SiO2 used as an example for the SIMPLE-NN machine learning model. Dataset includes three types of crystals: quartz, cristobalite and tridymite; amorphous; and liquid phase SiO2. Structures with distortion from compression, monoaxial strain and shear strain were also included in the training set."}],["DS_3tspv1150ejj_0",{"n":"SN2_JCTC_2019","d":"The SN2 dataset was generated as a partner benchmark dataset, along with the 'solvated protein fragments' dataset, for measuring the performance of machine learning models, in particular PhysNet, at describing chemical reactions, long-range interactions, and condensed phase systems. SN2 probes chemical reactions of methyl halides with halide anions, i.e. X- + CH3Y -> CH3X +  Y-, and contains structures, for all possible combinations of X,Y = F, Cl, Br, I. The dataset also includes various structures for several smaller molecules that can be formed in fragmentation reactions, such as CH3X, HX, CHX or CH2X- as well as geometries for H2, CH2, CH3+ and XY interhalogen compounds. In total, the dataset provides reference energies, forces, and dipole moments for 452709 structurescalculated at the DSD-BLYP-D3(BJ)/def2-TZVP level of theory using ORCA 4.0.1."}],["DS_kg0dv12aiq97_0",{"n":"SPICE_2023","d":"SPICE (Small-Molecule/Protein Interaction Chemical Energies) is a collection of quantum mechanical data for training potential functions. The emphasis is particularly on simulating drug-like small molecules interacting with proteins. Subsets of the dataset include the following: dipeptides: these provide comprehensive sampling of the covalent interactions found in proteins; solvated amino acids: these provide sampling of protein-water and water-water interactions; PubChem molecules: These sample a very wide variety of drug-like small molecules; monomer and dimer structures from DES370K: these provide sampling of a wide variety of non-covalent interactions; ion pairs: these provide further sampling of Coulomb interactions over a range of distances."}],["DS_a3v4bu6mts6b_0",{"n":"Si-H-GAP_reference","d":"A reference set of configurations of hydrogenated liquid and amorphous silicon from the datasets for Si-H-GAP. These configurations were used to evaluate training on a GAP model."}],["DS_8arvjtldu4fw_0",{"n":"Si-H-GAP_training","d":"A set of training configurations of hydrogenated liquid and amorphous silicon from the datasets for Si-H-GAP. Includes virial sigmas used for configurations used in the corresponding publication (virial-sigma-paper) as well as an alternate configuration defined by doubled virial sigma prefactors (from 0.025 to 0.05)."}],["DS_vywqeohskg2j_0",{"n":"Si-H-GAP_validation","d":"A set of validation configurations of hydrogenated liquid and amorphous silicon from the datasets for Si-H-GAP. These configurations served to augment the reference set as a final benchmark for NEP model performance."}],["DS_dqascvm2baak_0",{"n":"Si_Al_Ti_Seko_PRB_2019_test","d":"Training sets from Si_Al_Ti_Seko_PRB_2019. This dataset is compiled of 10,000 selected structures from the ICSD, divided into training and test sets. The dataset was generated for the purpose of training a MLIP with introduced high-order linearly independent rotational invariants up to the sixth order based on spherical harmonics. DFT calculations were carried out with VASP using the PBE cross-correlation functional and an energy cutoff of 400 eV."}],["DS_swqa99vqo249_0",{"n":"Si_Al_Ti_Seko_PRB_2019_train","d":"Test sets from Si_Al_Ti_Seko_PRB_2019. This dataset is compiled of 10,000 selected structures from the ICSD, divided into training and test sets. The dataset was generated for the purpose of training a MLIP with introduced high-order linearly independent rotational invariants up to the sixth order based on spherical harmonics. DFT calculations were carried out with VASP using the PBE cross-correlation functional and an energy cutoff of 400 eV."}],["DS_paehju6qhaym_0",{"n":"Si_JCP_2017","d":"A dataset of 64-atom silicon configurations in four phases: cubic-diamond, (beta)-tin, R8, and liquid. MD simulations are run at 300, 600 and 900 K for solid phases; up to 2500 K for the L phase. All relaxations performed at zero pressure. Additional configurations prepared by random distortion of crystal structures. VASP was used with a PAW pseudopotential and PBE exchange correlation. k-point mesh was optimized for energy convergence of 0.5 meV/atom and stress convergence of 0.1 kbar. The plane wave energy cutoff was set to 300 eV. To reduce the correlation between data points MD, data were thinned by using one of every 100 consecutive structures from the MD simulations at 300 K and one of every 20 structures from higher temperature MD simulations."}],["DS_u9wd92plbetq_0",{"n":"Si_PRX_GAP","d":"The original DFT training data for the general-purpose silicon interatomic potential described in the associated publication. The kinds of configuration that we include are chosen using intuition and past experience to guide what needs to be included to obtain good coverage pertaining to a range of properties."}],["DS_14m394gnh3ae_0",{"n":"Silica_NPJCM_2022","d":"This dataset was created for the purpose of training an MLIP for silica (SiO2). For initial DFT computations, GPAW (in combination with ASE) was used with LDA, PBE and PBEsol functionals; and VASP with the SCAN functional. All calculations used the projector augmented-wave method. After comparison, it was found that SCAN performed best, and all values were recalculated using SCAN. An energy cut-off of 900 eV and a k-spacing of 0.23 \u00c5-1 were used."}],["DS_h1i3plesx7bc_0",{"n":"Sn-SCAN_PRM_2023","d":"Approximately 6,500 configurations of Sn, including Sn8, Sn16 and Sn32, used in developing a deep potential that predicts the phase diagram of Sn."}],["DS_a0bxs66goqvv_0",{"n":"TSFF_PLOS_2022","d":"One configuration of an enzyme: training data for a quantum-guided molecular mechanics model."}],["DS_rgtu2lkgv5rq_0",{"n":"Ta_Linear_JCP2014","d":"This data set was originally used to generate a linear SNAP potential for solid and liquid tantalum as published in Thompson, A.P. et. al, J. Comp. Phys. 285 (2015) 316-330."}],["DS_r6c6gt2s98xm_0",{"n":"Ta_PINN_2021","d":"A dataset consisting of the energies of supercells containing from 1 to 250 atoms. The supercells represent energy-volume relations for 8 crystal structures of Ta, 5 uniform deformation paths between pairs of structures, vacancies, interstitials, surfaces with low-index orientations, 4 symmetrical tilt grain boundaries, \u03b3-surfaces on the (110) and (211) fault planes, a [111] screw dislocation, liquid Ta, and several isolated clusters containing from 2 to 51 atoms. Some of the supercells contain static atomic configurations. However, most are snapshots of ab initio MD simulations at different densities, and temperatures ranging from 293 K to 3300 K. The BCC structure was sampled in the greatest detail, including a wide range of isotropic and uniaxial deformations."}],["DS_40zw467dnc6d_0",{"n":"Ta_PRM2019","d":"This dataset was designed to enable machine-learning of Ta elastic, thermal, and defect properties, as well as surface energetics, melting, and the structure of the liquid phase. The dataset was constructed by starting with the dataset from J. Byggm\u00e4star et al., Phys. Rev. B 100, 144105 (2019), then rescaling all of the configurations to the correct lattice spacing and adding in gamma surface configurations."}],["DS_yk3t004l8dpd_0",{"n":"TdS-PdV_Atari5200","d":"Approximately 45,000 configurations of metal oxides of Mg, Ag, Pt, Cu and Zn, with initial training structures taken from the Materials Project database."}],["DS_jn819esw58ah_0",{"n":"TiMoS_alloys_CMS2021","d":"Training set (DFT output) for CE models and MC simulation output for the manuscript 'Phase behaviour of (Ti:Mo)S2binary alloys arising from electron-lattice coupling'. The DFT calculations are performed using VASP 5.4.3, compiled with intel MPI and Intel MKL support."}],["DS_kvjft3au55qb_0",{"n":"TiO2_CMS2016","d":"TiO2 dataset that was designed to build atom neural network potentials (ANN) by Artrith et al. using the AENET package. This dataset includes various crystalline phases of TiO2 and MD data that are extracted from ab inito calculations. The dataset includes 7815 structures with 165,229 atomic environments in the stochiometric ratio of 66% O to 34% Ti."}],["DS_ngso7es93qnj_0",{"n":"TiZrHfTa_APS2021","d":"A dataset used to train machine-learning interatomic potentials (moment tensor potentials) for multicomponent alloys to ab initio data in order to investigate the disordered body-centered cubic (bcc) TiZrHfTax system with varying Ta concentration. "}],["DS_aaocsvp2x40m_0",{"n":"Ti_NPJCM_2021","d":"Approximately 7,400 configurations of titanium used for training a deep potential using the DeePMD-kit molecular dynamics package and DP-GEN training scheme."}],["DS_zzfaosyakwom_0",{"n":"Transition1x-test","d":"The test split of the Transition1x dataset. Transition1x is a benchmark dataset containing 9.6 million Density Functional Theory (DFT) calculations of forces and energies of molecular configurations on and around reaction pathways at the \u03c9B97x/6-31 G(d) level of theory. The configurations contained in this dataset allow a better representation of features in transition state regions when compared to other benchmark datasets -- in particular QM9 and ANI1x."}],["DS_ktku4cml3al7_0",{"n":"Transition1x-validation","d":"The validation split of the Transition1x dataset. Transition1x is a benchmark dataset containing 9.6 million Density Functional Theory (DFT) calculations of forces and energies of molecular configurations on and around reaction pathways at the \u03c9B97x/6-31 G(d) level of theory. The configurations contained in this dataset allow a better representation of features in transition state regions when compared to other benchmark datasets -- in particular QM9 and ANI1x."}],["DS_sn0csplf32qj_0",{"n":"Transition1x_train","d":"The training split of the Transition1x dataset. Transition1x is a benchmark dataset containing 9.6 million Density Functional Theory (DFT) calculations of forces and energies of molecular configurations on and around reaction pathways at the \u03c9B97x/6-31 G(d) level of theory. The configurations contained in this dataset allow a better representation of features in transition state regions when compared to other benchmark datasets -- in particular QM9 and ANI1x."}],["DS_14h4rvviya0k_0",{"n":"UNEP-v1_2023","d":"This dataset contains training data for UNEP-v1 (version 1 of Unified NeuroEvolution Potential), a model implemented in GPUMD. Included are 16 separate configuration sets for individual elemental metals, as well as their 120 unique pairs."}],["DS_ouwlietscprn_0",{"n":"V_PRM2019","d":"This dataset was designed to enable machine-learning of V elastic, thermal, and defect properties, as well as surface energetics, melting, and the structure of the liquid phase. The dataset was constructed by starting with the dataset from J. Byggm\u00e4star et al., Phys. Rev. B 100, 144105 (2019), then rescaling all of the configurations to the correct lattice spacing and adding in gamma surface configurations."}],["DS_kk4a8u1eo42m_0",{"n":"W-14","d":"158,000 diverse atomic environments of elemental tungsten.Includes DFT-PBE energies, forces and stresses for tungsten; periodic unit cells in the range of 1-135 atoms, including bcc primitive cell, 128-atom bcc cell, vacancies, low index surfaces, gamma-surfaces, and dislocation cores."}],["DS_yq2whjjndyq5_0",{"n":"WBe_PRB2019","d":"This data set was originally used to generate a multi-component linear SNAP potential for tungsten and beryllium as published in Wood, M. A., et. al. Phys. Rev. B 99 (2019) 184305. This data set was developed for the purpose of studying plasma material interactions in fusion reactors."}],["DS_wqezpu0pw9io_0",{"n":"WS22_acrolein","d":"Configurations of acrolein from WS22. The WS22 database combines Wigner sampling with geometry interpolation to generate 1.18 million molecular geometries equally distributed into 10 independent datasets of flexible organic molecules with varying sizes and chemical complexity. In addition to the potential energy and forces required to construct potential energy surfaces, the WS22 database provides several other quantum chemical properties, all obtained via single-point calculations for each molecular geometry. All quantum chemical calculations were performed with the Gaussian 09 program."}],["DS_vxtsip8qgm7t_0",{"n":"WS22_alanine","d":"Configurations of alanine from WS22. The WS22 database combines Wigner sampling with geometry interpolation to generate 1.18 million molecular geometries equally distributed into 10 independent datasets of flexible organic molecules with varying sizes and chemical complexity. In addition to the potential energy and forces required to construct potential energy surfaces, the WS22 database provides several other quantum chemical properties, all obtained via single-point calculations for each molecular geometry. All quantum chemical calculations were performed with the Gaussian 09 program."}],["DS_srva1ami36vs_0",{"n":"WS22_dmabn","d":"Configurations of dmabn from WS22. The WS22 database combines Wigner sampling with geometry interpolation to generate 1.18 million molecular geometries equally distributed into 10 independent datasets of flexible organic molecules with varying sizes and chemical complexity. In addition to the potential energy and forces required to construct potential energy surfaces, the WS22 database provides several other quantum chemical properties, all obtained via single-point calculations for each molecular geometry. All quantum chemical calculations were performed with the Gaussian 09 program."}],["DS_s3qsg4c3hyk6_0",{"n":"WS22_nitrophenol","d":"Configurations of nitrophenol from WS22. The WS22 database combines Wigner sampling with geometry interpolation to generate 1.18 million molecular geometries equally distributed into 10 independent datasets of flexible organic molecules with varying sizes and chemical complexity. In addition to the potential energy and forces required to construct potential energy surfaces, the WS22 database provides several other quantum chemical properties, all obtained via single-point calculations for each molecular geometry. All quantum chemical calculations were performed with the Gaussian 09 program."}],["DS_ukw33o4tvaiy_0",{"n":"WS22_o-hbdi","d":"Configurations of o-hbdi from WS22. The WS22 database combines Wigner sampling with geometry interpolation to generate 1.18 million molecular geometries equally distributed into 10 independent datasets of flexible organic molecules with varying sizes and chemical complexity. In addition to the potential energy and forces required to construct potential energy surfaces, the WS22 database provides several other quantum chemical properties, all obtained via single-point calculations for each molecular geometry. All quantum chemical calculations were performed with the Gaussian 09 program."}],["DS_jz8yny1m7xer_0",{"n":"WS22_sma","d":"Configurations of sma from WS22. The WS22 database combines Wigner sampling with geometry interpolation to generate 1.18 million molecular geometries equally distributed into 10 independent datasets of flexible organic molecules with varying sizes and chemical complexity. In addition to the potential energy and forces required to construct potential energy surfaces, the WS22 database provides several other quantum chemical properties, all obtained via single-point calculations for each molecular geometry. All quantum chemical calculations were performed with the Gaussian 09 program."}],["DS_ngd7bajt6nvx_0",{"n":"WS22_thymine","d":"Configurations of o-hbdi from WS22. The WS22 database combines Wigner sampling with geometry interpolation to generate 1.18 million molecular geometries equally distributed into 10 independent datasets of flexible organic molecules with varying sizes and chemical complexity. In addition to the potential energy and forces required to construct potential energy surfaces, the WS22 database provides several other quantum chemical properties, all obtained via single-point calculations for each molecular geometry. All quantum chemical calculations were performed with the Gaussian 09 program."}],["DS_ztubsom32o4f_0",{"n":"WS22_toluene","d":"Configurations of toluene from WS22. The WS22 database combines Wigner sampling with geometry interpolation to generate 1.18 million molecular geometries equally distributed into 10 independent datasets of flexible organic molecules with varying sizes and chemical complexity. In addition to the potential energy and forces required to construct potential energy surfaces, the WS22 database provides several other quantum chemical properties, all obtained via single-point calculations for each molecular geometry. All quantum chemical calculations were performed with the Gaussian 09 program."}],["DS_0upwmrh3apql_0",{"n":"WS22_urea","d":"Configurations of urea from WS22. The WS22 database combines Wigner sampling with geometry interpolation to generate 1.18 million molecular geometries equally distributed into 10 independent datasets of flexible organic molecules with varying sizes and chemical complexity. In addition to the potential energy and forces required to construct potential energy surfaces, the WS22 database provides several other quantum chemical properties, all obtained via single-point calculations for each molecular geometry. All quantum chemical calculations were performed with the Gaussian 09 program."}],["DS_w3sximoit8bj_0",{"n":"WS22_urocanic","d":"Configurations of urocanic from WS22. The WS22 database combines Wigner sampling with geometry interpolation to generate 1.18 million molecular geometries equally distributed into 10 independent datasets of flexible organic molecules with varying sizes and chemical complexity. In addition to the potential energy and forces required to construct potential energy surfaces, the WS22 database provides several other quantum chemical properties, all obtained via single-point calculations for each molecular geometry. All quantum chemical calculations were performed with the Gaussian 09 program."}],["DS_0od3fns1ap8a_0",{"n":"W_LML-retrain_bulk_MD_test","d":"Test set from W_LML-retrain dataset, containing bulk tungsten calculations. The W_LML-retrain dataset contains DFT calculations used in testing a linear-in-descriptor machine learning potential that accounts for dislocation-defect interactions in tungsten. Density functional simulations were performed using VASP. The PBE generalised gradient approximation was used to describe effects of electron exchange and correlation together with a projector augmented wave (PAW) basis set with a cut-off energy of 550 eV. Occupancies were smeared with a Methfessel-Paxton scheme of order one with a 0.1 eV smearing width. The Brillouin zone was sampled with a Monkhorst-Pack k-point grid for the 2D cluster simulations periodic along the dislocation line and a single k-point was used for the calculations with 3D spherical QM regions. The values of these parameters were chosen after a series of convergence tests on forces with a tolerance of a few meV/\u00c5."}],["DS_gh9s2sopu064_0",{"n":"W_PRB2019","d":"This dataset was originally designed to fit a GAP potential with a specific focus on properties relevant for simulations of radiation-induced collision cascades and the damage they produce, including a realistic repulsive potential for short-range many-body cascade dynamics and a good description of the liquid phase."}],["DS_xrfpxtaioi75_0",{"n":"Yttrium-catalyzed_benzylic_C-H_alkylations_of_alkylpyridines_with_olefins","d":"This data was assembled to investigate rare-earth-catalyzed benzylic C(sp3)-H addition of pyridines to olefins. All calculations were performed with the Gaussian 09 software package. The B3PW91 functional was used for geometric optimization without any symmetric constraints. Each optimized structure was subsequently analyzed by harmonic vibrational frequencies at the same level of theory for characterization of a minimum (NImag = 0) or a transition state (NImag = 1) to obtain the thermodynamic data. The 6-31G(d) basis set was used for C, H, and N atoms, and Stuttgart/Dresden relativistic effective core potentials (RECPs) as well as the associated valence basis sets were used for the Y atom. To obtain more accurate energies, single-point energy calculations were performed with a larger basis set. In such single-point calculations, the M06-L functional, which often shows good performance in the treatment of transition-metal systems, was used together with the CPCM solvation model for consideration of the toluene solvation effect. The same basis set together with associated pseudopotentials as in geometry optimization was used for the Y atom, and the 6-311+G(d,p) basis set was used for the remaining atoms."}],["DS_sh7jt3ptmde4_0",{"n":"ZIF-4_Amorphous_Zeolitic_Imidazolate_Frameworks_2023","d":"This dataset contains four trajectories of amorphous zeolitic imidazolate frameworks (ZIF-4),  liquids calculated at four different volumes and at temperatures of 1500K and 1750K; and three trajectories of the ZIF-4 crystal: one at 300K and two at 1500K. Data was generated at the DFT-PBE-D3 level of theory."}],["DS_9in0wrvt6qg2_0",{"n":"Zeo-1_SD_2022","d":"130,000 configurations of zeolite from the Database of Zeolite Structures. Calculations performed using Amsterdam Modeling Suite software."}],["DS_58y020ce6b6j_0",{"n":"Zn_MTP_CMS2023","d":"A training dataset of diverse atomic configurations of Zn, varying in aggregation states, crystal structures, defect types, and sizes. The aim was to derive a potential capable of accurately describing a broad spectrum of local atomic configurations in Zn."}],["DS_6woak771jubv_0",{"n":"Zr_Sn_JNM_2024","d":"This dataset contains data from density functional theory calculations of various atomic configurations of pure Zr, pure Sn, and Zr-Sn alloys with different structures, defects, and compositions. Energies, forces, and stresses are calculated at the DFT level of theory. Includes 23,956 total configurations."}],["DS_70btumen3361_0",{"n":"a-AlOx_JCP_2020","d":"This dataset was used for the training of an MLIP for amorphous alumina (a-AlOx). Two configurations sets correspond to i) the actual training data and ii) additional reference data. Ab initio calculations were performedwith the Vienna Ab initio Simulation Package. The projector augmented wave method was used to treat the atomic core electrons,and the Perdew-Burke-Ernzerhof functional within the generalized gradient approximation was used to describe the electron-electron interactions. The cutoff energy for the plane-wave basis set was set to 550 eV during the ab initio calculation. The obtained reference database includes the DFT energies of 41,203 structures. The supercell size of the AlOx reference structures varied from 24 to 132 atoms. K-point values are given for structures with: Al0, Al12, Al24, Al48 and Al192."}],["DS_bmjfal3bj4ah_0",{"n":"aC_JCP_2023","d":"The amorphous carbon dataset was generated using ab initio calculations with VASP software. We utilized the LDA exchange-correlation functional and the PAW potential for carbon. Melt-quench simulations were performed to create amorphous and liquid-state structures. A simple cubic lattice of 216 carbon atoms was chosen as the initial state. Simulations were conducted at densities of 1.5, 1.7, 2.0, 2.2, 2.4, 2.6, 2.8, 3.0, 3.2, 3.4, and 3.5 g/cm3 to produce a variety of structures. The NVT ensemble was employed for all melt-quench simulations, and the density was adjusted by modifying the size of the simulation cell. A time step of 1 fs was used for the simulations. For all densities, only the \u0393 points were sampled in the k-space. To increase structural diversity, six independent simulations were performed.In the melt-quench simulations, the temperature was raised from 300 K to 9000 K over 2 ps to melt carbon. Equilibrium molecular dynamics (MD) was conducted at 9000 K for 3 ps to create a liquid state, followed by a decrease in temperature to 5000 K over 2 ps, with the system equilibrating at that temperature for 2 ps. Finally, the temperature was lowered from 5000 K to 300 K over 2 ps to generate an amorphous structure.During the melt-quench simulation, 30 snapshots were taken from the equilibrium MD trajectory at 9000 K, 100 from the cooling process between 9000 and 5000 K, 25 from the equilibrium MD trajectory at 5000 K, and 100 from the cooling process between 5000 and 300 K. This yielded a total of 16,830 data points.Data for diamond structures containing 216 atoms at densities of 2.4, 2.6, 2.8, 3.0, 3.2, 3.4, and 3.5 g/cm3 were also prepared. Further data on the diamond structure were obtained from 80 snapshots taken from the 2 ps equilibrium MD trajectory at 300 K, resulting in 560 data points.To validate predictions for larger structures, we generated data for 512-atom systems using the same procedure as for the 216-atom systems. A single simulation was conducted for each density. The number of data points was 2,805 for amorphous and liquid states"}],["DS_sany2behzwv8_0",{"n":"aC_JCP_2023_test","d":"Test split from the 216-atom amorphous portion of the aC_JCP_2023 dataset. The amorphous carbon dataset was generated using ab initio calculations with VASP software. We utilized the LDA exchange-correlation functional and the PAW potential for carbon. Melt-quench simulations were performed to create amorphous and liquid-state structures. A simple cubic lattice of 216 carbon atoms was chosen as the initial state. Simulations were conducted at densities of 1.5, 1.7, 2.0, 2.2, 2.4, 2.6, 2.8, 3.0, 3.2, 3.4, and 3.5 g/cm3 to produce a variety of structures. The NVT ensemble was employed for all melt-quench simulations, and the density was adjusted by modifying the size of the simulation cell. A time step of 1 fs was used for the simulations. For all densities, only the \u0393 points were sampled in the k-space. To increase structural diversity, six independent simulations were performed.In the melt-quench simulations, the temperature was raised from 300 K to 9000 K over 2 ps to melt carbon. Equilibrium molecular dynamics (MD) was conducted at 9000 K for 3 ps to create a liquid state, followed by a decrease in temperature to 5000 K over 2 ps, with the system equilibrating at that temperature for 2 ps. Finally, the temperature was lowered from 5000 K to 300 K over 2 ps to generate an amorphous structure.During the melt-quench simulation, 30 snapshots were taken from the equilibrium MD trajectory at 9000 K, 100 from the cooling process between 9000 and 5000 K, 25 from the equilibrium MD trajectory at 5000 K, and 100 from the cooling process between 5000 and 300 K. This yielded a total of 16,830 data points.Data for diamond structures containing 216 atoms at densities of 2.4, 2.6, 2.8, 3.0, 3.2, 3.4, and 3.5 g/cm3 were also prepared. Further data on the diamond structure were obtained from 80 snapshots taken from the 2 ps equilibrium MD trajectory at 300 K, resulting in 560 data points.To validate predictions for larger structures, we generated data for 512-atom systems using the same procedure as for the 216-atom systems. A single simulation was conducted for each density. The number of data points was 2,805 for amorphous and liquid states"}],["DS_20h8o5d0ltx1_0",{"n":"aC_JCP_2023_train","d":"Test split from the 216-atom amorphous portion of the aC_JCP_2023 dataset. The amorphous carbon dataset was generated using ab initio calculations with VASP software. We utilized the LDA exchange-correlation functional and the PAW potential for carbon. Melt-quench simulations were performed to create amorphous and liquid-state structures. A simple cubic lattice of 216 carbon atoms was chosen as the initial state. Simulations were conducted at densities of 1.5, 1.7, 2.0, 2.2, 2.4, 2.6, 2.8, 3.0, 3.2, 3.4, and 3.5 g/cm3 to produce a variety of structures. The NVT ensemble was employed for all melt-quench simulations, and the density was adjusted by modifying the size of the simulation cell. A time step of 1 fs was used for the simulations. For all densities, only the \u0393 points were sampled in the k-space. To increase structural diversity, six independent simulations were performed.In the melt-quench simulations, the temperature was raised from 300 K to 9000 K over 2 ps to melt carbon. Equilibrium molecular dynamics (MD) was conducted at 9000 K for 3 ps to create a liquid state, followed by a decrease in temperature to 5000 K over 2 ps, with the system equilibrating at that temperature for 2 ps. Finally, the temperature was lowered from 5000 K to 300 K over 2 ps to generate an amorphous structure.During the melt-quench simulation, 30 snapshots were taken from the equilibrium MD trajectory at 9000 K, 100 from the cooling process between 9000 and 5000 K, 25 from the equilibrium MD trajectory at 5000 K, and 100 from the cooling process between 5000 and 300 K. This yielded a total of 16,830 data points.Data for diamond structures containing 216 atoms at densities of 2.4, 2.6, 2.8, 3.0, 3.2, 3.4, and 3.5 g/cm3 were also prepared. Further data on the diamond structure were obtained from 80 snapshots taken from the 2 ps equilibrium MD trajectory at 300 K, resulting in 560 data points.To validate predictions for larger structures, we generated data for 512-atom systems using the same procedure as for the 216-atom systems. A single simulation was conducted for each density. The number of data points was 2,805 for amorphous and liquid states"}],["DS_08feepssbuq5_0",{"n":"adatoms_on_single-layer_graphene_PRR2021","d":"This dataset consists of graphene superlattices with tungsten adatoms with properties calculated at the DFT level of theory. The authors modeled the placement of tungsten adatoms on a graphene monolayer. The resulting superlattice structures were then used to calculate electronic band structure and phonon dispersion relations. The dataset was used to investigate the effect of adatom placement on electronic band structure and phonon dispersion relations of graphene superlattices. The creation of the dataset involved the following steps: 1. Selection of the graphene monolayer as the starting point for the superlattice construction. 2. Placement of tungsten adatoms in the center of the unit cell 3. Calculation of the electronic structure and other properties of the resulting superlattice using DFT. 4. Generation of a set of reduced Brillouin zones representing the symmetry of the superlattice. 5. Calculation of the electronic band structure and phonon dispersion relations for each superlattice structure in the dataset. "}],["DS_agiti2oe5bqb_0",{"n":"alpha_brass_nanoparticles","d":"53,841 structures of alpha-brass (less than 40% Zinc). Includes atomic forces and total energy. Calculated using VASP at the DFT level of theory."}],["DS_xzaglubh0trq_0",{"n":"cG-SchNet","d":"Configurations from a cG-SchNet trained on a subset of the QM9dataset. Model was trained with the intention of providing molecules withspecified functional groups or motifs, relying on sampling of molecularfingerprint data. Relaxation data for the generated molecules is computedusing ORCA software. Configuration sets include raw data fromcG-SchNet-generated configurations, with models trained on several differenttypes of target data and DFT relaxation data as a separate configurationset. Includes approximately 80,000 configurations."}],["DS_w59koijcczkz_0",{"n":"calcium_ferrites_as_cathodes_ca4fe9o17","d":"Dataset for \"Appraisal of calcium ferrites as cathodes for calcium rechargeable batteries: DFT, synthesis, characterization and electrochemistry of Ca4Fe9O17\" created to explore Fe-based cathode materials for Ca-ion batteries. Structures include CaFe(2+n)O(4+n), where 0 < n < 3."}],["DS_s00e64z80ujy_0",{"n":"cathode_materials_for_rechargeable_Ca_batteries_CM2021","d":"Data from the publication \"Enlisting Potential Cathode Materials for Rechargeable Ca Batteries\". The development of rechargeable batteries based on a Ca metal anode demands the identification of suitable cathode materials. This work investigates the potential application of a variety of compounds, which are selected from the In-organic Crystal Structural Database (ICSD) considering 3d-transition metal oxysulphides, pyrophosphates, silicates, nitrides, and phosphates with a maximum of four different chemical elements in their composition. Cathode perfor-mance of CaFeSO, CaCoSO, CaNiN, Ca3MnN3, Ca2Fe(Si2O7), CaM(P2O7) (M = V, Cr, Mn, Fe, Co), CaV2(P2O7)2, Ca(VO)2(PO4)2 and \u03b1-VOPO4 is evaluated throughout the calculation of operation voltages, volume changes associated to the redox reaction and mobility of Ca2+ ions. Some materials exhibit attractive specific capacities and intercalation voltages combined with energy barriers for Ca migration around 1 eV (CaFeSO, Ca2FeSi2O7 and CaV2(P2O7)2). Based on the DFT results, \u03b1I-VOPO4 is identified as a potential Ca-cathode with a maximum theoretical specific capacity of 312 mAh/g, an average intercalation voltage of 2.8 V and calculated energy barriers for Ca migration below 0.65 eV (GGA functional)."}],["DS_mf8sn11cn6wa_0",{"n":"datasets_for_magnetic_MTP_NatSR2024_training","d":"This dataset comprises a training dataset for magnetic multi-component machine-learning potentials for Fe-Al systems, including different concentrations of Fe and Al (Al concentrations from 0%-50%), with fully equilibrated and perturbed atomic positions, lattice vectors and magnetic moments represented."}],["DS_wu6xd9i8cf7i_0",{"n":"datasets_for_magnetic_MTP_NatSR2024_verification","d":"This is the verification dataset (see companion training dataset: datasets_for_magnetic_MTP_NatSR2024_training) used in training a magnetic multi-component machine-learning potential for Fe-Al systems. The configurations from the verification set include different levels of magnetic moment perturbation than configurations from the training set. For this reason, the authors refer to this dataset as a \"verification set\", rather than a \"validation set\"."}],["DS_k059wtxqsksu_0",{"n":"defected_phosphorene_ACS_2023","d":"This dataset contains pristine monolayer phosphorene as well as structures with monovacancies which were used to train an artificial neural network (ANN) for use with a high-dimensional neural network potentials molecular dynamics (HDNNP-MD) simulation. The publication investigates the mechanism and rates of the processes of defect diffusion, as well as monovacancy-to-divacancy defect coalescence."}],["DS_q6e3bvq4y67a_0",{"n":"discrepencies_and_error_metrics_NPJ_2023_enhanced_validation_set","d":"Structures from discrepencies_and_error_metrics_NPJ_2023 validation set, enhanced by inclusion of rare events. The full discrepencies_and_error_metrics_NPJ_2023 dataset includes the original mlearn_Si_train dataset, modified with the purpose of developing models with better diffusivity scores by replacing ~54% of the data with structures containing migrating interstitials. The enhanced validation set contains 50 total structures, consisting of 20 structures randomly selected from the 120 replaced structures of the original training dataset, 11 snapshots with vacancy rare events (RE) from AIMD simulations, and 19 snapshots with interstitial RE from AIMD simulations. We also construct interstitial-RE and vacancy-RE testing sets, each consisting of 100 snapshots of atomic configurations with a single migrating vacancy or interstitial, respectively, from AIMD simulations at 1230 K."}],["DS_nublbp38wse0_0",{"n":"discrepencies_and_error_metrics_NPJ_2023_interstitial_enhanced_training_set","d":"Structures from discrepencies_and_error_metrics_NPJ_2023 training set, enhanced by inclusion of interstitials. The full discrepencies_and_error_metrics_NPJ_2023 dataset includes the original mlearn_Si_train dataset, modified with the purpose of developing models with better diffusivity scores by replacing ~54% of the data with structures containing migrating interstitials. The enhanced validation set contains 50 total structures, consisting of 20 structures randomly selected from the 120 replaced structures of the original training dataset, 11 snapshots with vacancy rare events (RE) from AIMD simulations, and 19 snapshots with interstitial RE from AIMD simulations. We also construct interstitial-RE and vacancy-RE testing sets, each consisting of 100 snapshots of atomic configurations with a single migrating vacancy or interstitial, respectively, from AIMD simulations at 1230 K."}],["DS_dhe9aqs9q1wf_0",{"n":"discrepencies_and_error_metrics_NPJ_2023_interstitial_re_testing_set","d":"Structures from discrepencies_and_error_metrics_NPJ_2023 test set; these include an interstitial. The full discrepencies_and_error_metrics_NPJ_2023 dataset includes the original mlearn_Si_train dataset, modified with the purpose of developing models with better diffusivity scores by replacing ~54% of the data with structures containing migrating interstitials. The enhanced validation set contains 50 total structures, consisting of 20 structures randomly selected from the 120 replaced structures of the original training dataset, 11 snapshots with vacancy rare events (RE) from AIMD simulations, and 19 snapshots with interstitial RE from AIMD simulations. We also construct interstitial-RE and vacancy-RE testing sets, each consisting of 100 snapshots of atomic configurations with a single migrating vacancy or interstitial, respectively, from AIMD simulations at 1230 K."}],["DS_qxd7wv9yabtp_0",{"n":"discrepencies_and_error_metrics_NPJ_2023_vacancy_enhanced_training_set","d":"Structures from discrepencies_and_error_metrics_NPJ_2023 training set; includes some structures with vacancies. The full discrepencies_and_error_metrics_NPJ_2023 dataset includes the original mlearn_Si_train dataset, modified with the purpose of developing models with better diffusivity scores by replacing ~54% of the data with structures containing migrating interstitials. The enhanced validation set contains 50 total structures, consisting of 20 structures randomly selected from the 120 replaced structures of the original training dataset, 11 snapshots with vacancy rare events (RE) from AIMD simulations, and 19 snapshots with interstitial RE from AIMD simulations. We also construct interstitial-RE and vacancy-RE testing sets, each consisting of 100 snapshots of atomic configurations with a single migrating vacancy or interstitial, respectively, from AIMD simulations at 1230 K."}],["DS_la08goe2lz0g_0",{"n":"discrepencies_and_error_metrics_NPJ_2023_vacancy_re_testing_set","d":"Structures from discrepencies_and_error_metrics_NPJ_2023 test set; these include a single migrating vacancy. The full discrepencies_and_error_metrics_NPJ_2023 dataset includes the original mlearn_Si_train dataset, modified with the purpose of developing models with better diffusivity scores by replacing ~54% of the data with structures containing migrating interstitials. The enhanced validation set contains 50 total structures, consisting of 20 structures randomly selected from the 120 replaced structures of the original training dataset, 11 snapshots with vacancy rare events (RE) from AIMD simulations, and 19 snapshots with interstitial RE from AIMD simulations. We also construct interstitial-RE and vacancy-RE testing sets, each consisting of 100 snapshots of atomic configurations with a single migrating vacancy or interstitial, respectively, from AIMD simulations at 1230 K."}],["DS_k1k8iul6kgm2_0",{"n":"disordered_transition_metal_oxyfluorides_EA2021","d":"Data from \"On-the-fly assessment of diffusion barriers of disordered transition metal oxyfluorides using local descriptors\". The dataset contains the result of 48 Nudged Elastic Band calculations of Li(2-x)VO2F diffusion barriers. The NEB was performed with VASP, using projector augmented-wave (PAW) method to describe electron-ion interaction. The disordered rock salt cells were created using a 3 x 4 x 4 supercell containing 96 atoms (in case of no vacancies). PBE is used as XC functional while a rotationally invariant Hubbard U correction was applied to the d orbital of V with a U value of 3.25 eV."}],["DS_a2ftwj9b873a_0",{"n":"doped_CsPbI3_energetics_test","d":"The test set from the doped CsPbI3 energetics dataset. This dataset was created to explore the effect of Cd and Pb substitutions on the structural stability of inorganic lead halide perovskite CsPbI3. CsPbI3 undergoes a direct to indirect band-gap phase transition at room temperature. The dataset contains configurations of CsPbI3 with low levels of Cd and Zn, which were used to train a GNN model to predict the energetics of structures with higher levels of substitutions."}],["DS_doj9b688juif_0",{"n":"doped_CsPbI3_energetics_train_validate","d":"The training + validation set from the doped CsPbI3 energetics dataset. This dataset was created to explore the effect of Cd and Pb substitutions on the structural stability of inorganic lead halide perovskite CsPbI3. CsPbI3 undergoes a direct to indirect band-gap phase transition at room temperature. The dataset contains configurations of CsPbI3 with low levels of Cd and Zn, which were used to train a GNN model to predict the energetics of structures with higher levels of substitutions."}],["DS_swl40hdnt479_0",{"n":"electrode_materials_for_ca-based_rechargeable_batteries","d":"Dataset for \"Analysis of minerals as electrode materials for Ca-based rechargeable batteries\". Includes DFT structures of pyroxenes, garnet and carbonates. Dataset was produced to pursue identification of Ca-based high specific energy cathode materials."}],["DS_1t2xs8bzygtp_0",{"n":"ferroelectricity_and_metallicity_in_BaTiO3_JMCC2021","d":"Dataset for \"Interplay between ferroelectricity and metallicity in BaTiO3\", exploring properties of ferroelectric barium titanate (BaTiO3), including the effects of electron and hole doping. Includes configuration sets for unit cells and supercells of BaTiO3."}],["DS_i23sbm1o45sj_0",{"n":"flexible_molecules_JCP2021","d":"Configurations of azobenzene featuring a cis to trans thermal inversion through three channels: inversion, rotation, and rotation assisted by inversion; and configurations of glycine as a simpler comparison molecule. All calculations were performed in FHI-aims software using the Perdew-Burke-Ernzerhof (PBE) exchange-correlation functional with the Tkatchenko-Scheffler (TS) method to account for van der Waals (vdW) interactions. The azobenzene sets contain calculations from several different MD simulations, including two long simulations initialized at 300 K; short simulations (300 steps) initialized at 300 K and shorter (.5fs) timestep; four simulations, two starting from each of cis and trans isomer, at 750 K (initialized at 3000 K); and simulations at 50 K (initialized at 300 K). The glycine isomerization set was built using one MD simulation starting from each of two different minima. Initializatin and simulation temperature were 500 K."}],["DS_cpznjcu51bvg_0",{"n":"glass-ceramic_lithium_thiophosphate_electrolytes_","d":"This database contains computationally generated atomic structures of glass-ceramics lithium thiophosphates (gc-LPS) with the general composition (Li2S)x(P2S5)1-x. Total energies and interatomic forces from density-functional theory (DFT) calculations are included. The DFT calculations used projector-augmented-wave (PAW) pseudopotentials and the Perdew-Burke-Ernzerhof (PBE) exchange-correlation functional as implemented in the Vienna Ab Initio Simulation Package (VASP) and a kinetic energy cutoff of 520 eV. The first Brillouin zone was sampled using VASP's fully automatic k-point scheme with a length parameter Rk = 25\u00c5. The gc-LPS structures were generated using a combination of different sampling methods. Initial amorphous structure models were generated with ab initio molecular dynamics (AIMD) simulations of supercells at 1200 K using a Nose-Hoover thermostat with a time step of 1 fs. To obtain near-ground-state structures as reference for the machine-learning potential, 150 evenly spaced snapshots were extracted from the AIMD trajectories that were reoptimized with DFT geometry optimizations at zero Kelvin. Additional structures were generated by scaling the lattice parameters of the crystalline LPS structures (see below) by \u00b115% and perturbing atomic positions in AIMD simulations as described above.The resulting database was used to train a specialized ANN potential for the sampling of structures along the Li2S-P2S5 composition line with a genetic-algorithm (GA) as implemented in the atomistic evolution (\u00e6vo) package, following a previously reported protocol. Starting from supercells of the ideal crystal structures, either Li and S atoms were removed with a ratio of 2:1, or P and S atoms were removed with a ratio of 2:5, and low-energy configurations were determined with GA sampling. A population size of 32 trials and a mutation rate of 10% were employed. The ANN potential was iteratively refined by including additional sampled structures in the training. For each composition, at least 10 lowest energy structure models identified with the ANN-GA approach were selected and fully relaxed with DFT.Also included in the present database are the XSF files of the previously reported crystalline phases LiPS3, Li2PS3, Li4P2S7, Li7P3S11, \u03b1-Li3PS4, \u03b2-Li3PS4, \u03b3-Li3PS4, and Li48P16S61. The crystal structures were obtained from the Inorganic Crystal Structure Database (ICSD). the Materials Project (MP) database, the Open Quantum Materials Database (OQMD), and the AFLOW database. The configuration names indicate the journal reference and the database."}],["DS_82x5bfiiyaij_0",{"n":"linear_magnetic_coefficient_in_Cr2O3_JPCM2024","d":"We establish the sign of the linear magnetoelectric (ME) coefficient, \u03b1, in chromia, Cr\u2082O\u2083. Cr\u2082O\u2083 is the prototypical linear ME material, in which an electric (magnetic) field induces a linearly proportional magnetization (polarization), and a single magnetic domain can be selected by annealing in combined magnetic (H) and electric (E) fields. Opposite antiferromagnetic domains have opposite ME responses, and which antiferromagnetic domain corresponds to which sign of response has previously been unclear. We use density functional theory (DFT) to calculate the magnetic response of a single antiferromagnetic domain of Cr\u2082O\u2083 to an applied in-plane electric field at 0 K. We find that the domain with nearest neighbor magnetic moments oriented away from (towards) each other has a negative (positive) in-plane ME coefficient, \u03b1\u22a5, at 0 K. We show that this sign is consistent with all other DFT calculations in the literature that specified the domain orientation, independent of the choice of DFT code or functional, the method used to apply the field, and whether the direct (magnetic field) or inverse (electric field) ME response was calculated. Next, we reanalyze our previously published spherical neutron polarimetry data to determine the antiferromagnetic domain produced by annealing in combined E and H fields oriented along the crystallographic symmetry axis at room temperature. We find that the antiferromagnetic domain with nearest-neighbor magnetic moments oriented away from (towards) each other is produced by annealing in (anti-)parallel E and H fields, corresponding to a positive (negative) axial ME coefficient, \u03b1\u2225, at room temperature. Since \u03b1\u22a5 at 0 K and \u03b1\u2225 at room temperature are known to be of opposite sign, our computational and experimental results are consistent. This dataset contains the input data to reproduce the calculation of the magnetoelectric effect as plotted in Fig. 3 of the manuscript, for Elk, Vasp, and Quantum Espresso."}],["DS_t8goqf2uglhj_0",{"n":"local_polarization_in_oxygen-deficient_LaMnO3_PRR2020","d":"This dataset contains structural calculations of LaMnO3 carried out in Quantum ESPRESSO at the DFT-PBEsol+U level of theory. The dataset was built to explore strained and stoichiometric and oxygen-deficient LaMnO3."}],["DS_e94my2wrh074_0",{"n":"mbGDML_maldonado_2023","d":"Configurations of water, acetonitrile and methanol, simulated with ASE and modeled using a variety of software and methods: GAP, SchNet, GDML, ORCA and mbGDML. Forces and potential energy included; metadata includes kinetic energy and velocities."}],["DS_xmgn3ofqzon9_0",{"n":"mlearn_Cu_test","d":"A comprehensive DFT data set was generated for six elements - Li, Mo, Ni, Cu, Si, and Ge. These elements were chosen to span a variety of chemistries (main group metal, transition metal, and semiconductor), crystal structures (bcc, fcc, and diamond) and bonding types (metallic and covalent). This dataset comprises only the Cu configurations"}],["DS_3pv3hck35iy6_0",{"n":"mlearn_Cu_train","d":"A comprehensive DFT data set was generated for six elements - Li, Mo, Ni, Cu, Si, and Ge. These elements were chosen to span a variety of chemistries (main group metal, transition metal, and semiconductor), crystal structures (bcc, fcc, and diamond) and bonding types (metallic and covalent). This dataset comprises only the Cu configurations"}],["DS_pyrk84w3auvb_0",{"n":"mlearn_Ge_test","d":"A comprehensive DFT data set was generated for six elements - Li, Mo, Ni, Cu, Si, and Ge. These elements were chosen to span a variety of chemistries (main group metal, transition metal, and semiconductor), crystal structures (bcc, fcc, and diamond) and bonding types (metallic and covalent). This dataset comprises only the Ge configurations"}],["DS_ot3m0rxle8fs_0",{"n":"mlearn_Ge_train","d":"A comprehensive DFT data set was generated for six elements - Li, Mo, Ni, Cu, Si, and Ge. These elements were chosen to span a variety of chemistries (main group metal, transition metal, and semiconductor), crystal structures (bcc, fcc, and diamond) and bonding types (metallic and covalent). This dataset comprises only the Ge configurations"}],["DS_jf20mvwao5xi_0",{"n":"mlearn_Li_test","d":"A comprehensive DFT data set was generated for six elements - Li, Mo, Ni, Cu, Si, and Ge. These elements were chosen to span a variety of chemistries (main group metal, transition metal, and semiconductor), crystal structures (bcc, fcc, and diamond) and bonding types (metallic and covalent). This dataset comprises only the Li configurations"}],["DS_h7zao9ya9sd8_0",{"n":"mlearn_Li_train","d":"A comprehensive DFT data set was generated for six elements - Li, Mo, Ni, Cu, Si, and Ge. These elements were chosen to span a variety of chemistries (main group metal, transition metal, and semiconductor), crystal structures (bcc, fcc, and diamond) and bonding types (metallic and covalent). This dataset comprises only the Li configurations"}],["DS_l0b6iq3no012_0",{"n":"mlearn_Mo_test","d":"A comprehensive DFT data set was generated for six elements - Li, Mo, Ni, Cu, Si, and Ge. These elements were chosen to span a variety of chemistries (main group metal, transition metal, and semiconductor), crystal structures (bcc, fcc, and diamond) and bonding types (metallic and covalent). This dataset comprises only the Mo configurations"}],["DS_ytoet4uyc32k_0",{"n":"mlearn_Mo_train","d":"A comprehensive DFT data set was generated for six elements - Li, Mo, Ni, Cu, Si, and Ge. These elements were chosen to span a variety of chemistries (main group metal, transition metal, and semiconductor), crystal structures (bcc, fcc, and diamond) and bonding types (metallic and covalent). This dataset comprises only the Mo configurations"}],["DS_zjkz9664bapl_0",{"n":"mlearn_Ni_test","d":"A comprehensive DFT data set was generated for six elements - Li, Mo, Ni, Cu, Si, and Ge. These elements were chosen to span a variety of chemistries (main group metal, transition metal, and semiconductor), crystal structures (bcc, fcc, and diamond) and bonding types (metallic and covalent). This dataset comprises only the Ni configurations"}],["DS_lfyd4jv627cr_0",{"n":"mlearn_Ni_train","d":"A comprehensive DFT data set was generated for six elements - Li, Mo, Ni, Cu, Si, and Ge. These elements were chosen to span a variety of chemistries (main group metal, transition metal, and semiconductor), crystal structures (bcc, fcc, and diamond) and bonding types (metallic and covalent). This dataset comprises only the Ni configurations"}],["DS_lvjng6g41bwy_0",{"n":"mlearn_Si_test","d":"A comprehensive DFT data set was generated for six elements - Li, Mo, Ni, Cu, Si, and Ge. These elements were chosen to span a variety of chemistries (main group metal, transition metal, and semiconductor), crystal structures (bcc, fcc, and diamond) and bonding types (metallic and covalent). This dataset comprises only the Si configurations"}],["DS_eltotrjoqonr_0",{"n":"mlearn_Si_train","d":"A comprehensive DFT data set was generated for six elements - Li, Mo, Ni, Cu, Si, and Ge. These elements were chosen to span a variety of chemistries (main group metal, transition metal, and semiconductor), crystal structures (bcc, fcc, and diamond) and bonding types (metallic and covalent). This dataset comprises only the Si configurations"}],["DS_ots0ul69q842_0",{"n":"oxygen-vacancy_defects_in_Cu2O(111)","d":"This dataset investigates the effect of defects, such as copper and oxygen vacancies, in cuprous oxide films. Structures include oxygen vacancies formed in proximity of a reconstructed Cu2O(111) surface, where the outermost unsaturated copper atoms are removed, thus forming non-stoichiometric surface layers with copper vacancies. Surface and bulk properties are addressed by modelling a thick and symmetric slab consisting of 8 atomic layers and 736 atoms. Configuration sets include bulk, slab, vacancy and oxygen gas. Version v1"}],["DS_48yn17885mdi_0",{"n":"pure_magnesium_DFT_PRM2020","d":"This dataset provides DFT (as implemented in VASP) calculations for pure magnesium. Configuration sets include bulk, generalized stacking fault energies, stable stacking fault, decohesion, relaxed surfaces, dimer, corner and rod, and vacancy configurations of Mg."}],["DS_p9qjlc9l8scx_0",{"n":"q-AQUA","d":"The a-AQUA dataset was generated to address the need for a training set for a water PES that includes 2-body, 3-body and 4-body interactions calculated at the CCSD(T) level of theory. Structures were selected from the existing HBB2-pol and MB-pol datasets. For each water dimer structure, CCSD(T)/aug-cc-pVTZ calculations were performed with an additional 3s3p2d1f basis set; exponents equal to (0.9, 0.3, 0.1) for sp, (0.6, 0.2) for d, and 0.3 for f. This additional basis is placed at the center of mass (COM) of each dimer configuration. The basis set superposition error (BSSE) correction was determined with the counterpoise scheme. CCSD(T)/aug-cc-pVQZ calculations were then performed with the same additional basis set and BSSE correction. Final CCSD(T)/CBS energies were obtained by extrapolation over the CCSD(T)/aug-cc-pVTZ and CCSD(T)/aug-cc-pVQZ 2-b energies. All ab initio calculations were performed using Molpro package.Trimer structures were calculated at CCSD(T)-F12a/aug-cc-pVTZ with BSSE correction. Four-body structure calculations were performed at CCSD(T)-F12 level."}],["DS_8rafgy0ly6bt_0",{"n":"rMD17","d":"A dataset of 10 molecules (aspirin, azobenzene, benzene, ethanol, malonaldehyde, naphthalene, paracetamol, salicylic, toluene, uracil) with 100,000 structures calculated for each at the PBE/def2-SVP level of theory using ORCA. Based on the MD17 dataset, but with refined measurements."}],["DS_apdpxdjx082p_0",{"n":"reactive_hydrogen_ACS_2023","d":"This dataset contains structures of Cu, including Cu(111), Cu(100), Cu(110), and Cu(211). Slab settings are as follows: 3 x 3, 6-layered slabs for Cu(111), (100), and (110) surfaces; 1 x 3, 6-layered slabs for Cu(211) surface. Includes some structures representing interation of H2 with one of the Cu surfaces and some structures of Cu sampled at different temperatures."}],["DS_u7fu1meyxvpc_0",{"n":"reduced-perovskite_and_oxidized-marokite_oxides","d":"Dataset contains DFT calculations of oxygen-deficient perovskites from the Ca2Fe2O5-brownmillerite and Ca2Mn2O5 structures; and tunnel CaMn4O8, a derivative of the CaMn2O4-marokite with Ca vacancies. The dataset was produced to investigate the effects of oxygen introduction or Ca vacancy introduction in ternary transition metal oxides, as a means to assess potential new Ca-ion battery materials."}],["DS_cl44a8aa0fgb_0",{"n":"reduced-perovskite_and_oxidized-marokite_oxides","d":"Dataset contains DFT calculations of oxygen-deficient perovskites from the Ca2Fe2O5-brownmillerite and Ca2Mn2O5 structures; and tunnel CaMn4O8, a derivative of the CaMn2O4-marokite with Ca vacancies. The dataset was produced to investigate the effects of oxygen introduction or Ca vacancy introduction in ternary transition metal oxides, as a means to assess potential new Ca-ion battery materials."}],["DS_4e7a9g2kav0a_0",{"n":"sGDML_Aspirin_ccsd_NC2018_test","d":"The test set of a train/test pair from the aspirin dataset from sGDML. To create the coupled cluster datasets, the data used for training the models were created by running abinitio MD in the NVT ensemble using the Nos\u00e9-Hoover thermostat at 500 K during a 200 ps simulation with a resolution of 0.5 fs. Energies and forces were recalculated by all-electron coupled cluster with single, double and perturbative triple excitations (CCSD(T)). The Dunning correlation-consistent basis set CCSD/cc-pVDZ was used for aspirin. All calculations were performed with the Psi4 software suite."}],["DS_xy48avqcknnk_0",{"n":"sGDML_Aspirin_ccsd_NC2018_train","d":"The train set of a train/test pair from the aspirin dataset from sGDML. To create the coupled cluster datasets, the data used for training the models were created by running abinitio MD in the NVT ensemble using the Nos\u00e9-Hoover thermostat at 500 K during a 200 ps simulation with a resolution of 0.5 fs. Energies and forces were recalculated by all-electron coupled cluster with single, double and perturbative triple excitations (CCSD(T)). The Dunning correlation-consistent basis set CCSD/cc-pVDZ was used for aspirin. All calculations were performed with the Psi4 software suite."}],["DS_q9y1aat05u42_0",{"n":"sGDML_Benzene_DFT_NC2018","d":"The data used for training the DFT models were created running ab initio MD in the NVT ensemble using the Nos\u00e9-Hoover thermostat at 500 K during a 200 ps simulation with a resolution of 0.5 fs. Forces and energies were computed using all-electrons at the generalized gradient approximation level of theory with the Perdew-Burke-Ernzerhof (PBE) exchange-correlation functional, treating van der Waals interactions with the Tkatchenko-Scheffler (TS) method. All calculations were performed with FHI-aims. The final training data was generated by subsampling the full trajectory under preservation of the Maxwell-Boltzmann distribution for the energies."}],["DS_jol8dvjej92n_0",{"n":"sGDML_Benzene_ccsdt_NC2018_test","d":"The test set of a train/test pair from the benzene dataset from sGDML. To create the coupled cluster datasets, the data used for training the models were created by running ab initio MD in the NVT ensemble using the Nos\u00e9-Hoover thermostat at 500 K during a 200 ps simulation with a resolution of 0.5 fs. Energies and forces were recalculated using all-electron coupled cluster with single , double and perturbative triple excitations (CCSD(T)). The Dunning correlation-consistent basis set cc-pVDZ was used for benzene. All calculations were performed with the Psi4 software suite."}],["DS_8xk0v5v9dbx0_0",{"n":"sGDML_Benzene_ccsdt_NC2018_train","d":"The train set of a train/test pair from the benzene dataset from sGDML. To create the coupled cluster datasets, the data used for training the models were created by running ab initio MD in the NVT ensemble using the Nos\u00e9-Hoover thermostat at 500 K during a 200 ps simulation with a resolution of 0.5 fs. Energies and forces were recalculated using all-electron coupled cluster with single , double and perturbative triple excitations (CCSD(T)). The Dunning correlation-consistent basis set cc-pVDZ was used for benzene. All calculations were performed with the Psi4 software suite."}],["DS_iavvqpb14zqv_0",{"n":"sGDML_Ethanol_ccsdt_NC2018_test","d":"The test set of a train/test pair from the ethanol dataset from sGDML. To create the coupled cluster datasets, the data used for training the models were created by running ab initio MD in the NVT ensemble using the Nos\u00e9-Hoover thermostat at 500 K during a 200 ps simulation with a resolution of 0.5 fs. Energies and forces were recalculated using all-electron coupled cluster with single, double and perturbative triple excitations (CCSD(T)).The Dunning correlation-consistent basis set cc-pVTZ was used for ethanol. All calculations were performed with the Psi4 software suite."}],["DS_dalgmbg32lwz_0",{"n":"sGDML_Ethanol_ccsdt_NC2018_train","d":"The train set of a train/test pair from the ethanol dataset from sGDML. To create the coupled cluster datasets, the data used for training the models were created by running ab initio MD in the NVT ensemble using the Nos\u00e9-Hoover thermostat at 500 K during a 200 ps simulation with a resolution of 0.5 fs. Energies and forces were recalculated using all-electron coupled cluster with single, double and perturbative triple excitations (CCSD(T)).The Dunning correlation-consistent basis set cc-pVTZ was used for ethanol. All calculations were performed with the Psi4 software suite."}],["DS_yw3sqtaq40ll_0",{"n":"sGDML_Malonaldehyde_ccsdt_NC2018_test","d":"The test set of a train/test pair from the malonaldehyde dataset from sGDML. To create the coupled cluster datasets, the data used for training the models were created by running ab initio MD in the NVT ensemble using the Nos\u00e9-Hoover thermostat at 500 K during a 200 ps simulation with a resolution of 0.5 fs. Energies and forces were recalculated using all-electron coupled cluster with single, double and perturbative triple excitations (CCSD(T)). The Dunning correlation-consistent basis set cc-pVDZ was used for malonaldehyde. All calculations were performed with the Psi4 software suite."}],["DS_201vhvywll83_0",{"n":"sGDML_Malonaldehyde_ccsdt_NC2018_train","d":"The train set of a train/test pair from the malonaldehyde dataset from sGDML. To create the coupled cluster datasets, the data used for training the models were created by running ab initio MD in the NVT ensemble using the Nos\u00e9-Hoover thermostat at 500 K during a 200 ps simulation with a resolution of 0.5 fs. Energies and forces were recalculated using all-electron coupled cluster with single, double and perturbative triple excitations (CCSD(T)). The Dunning correlation-consistent basis set cc-pVDZ was used for malonaldehyde. All calculations were performed with the Psi4 software suite."}],["DS_ega58h2u9rwr_0",{"n":"sGDML_Toluene_ccsdt_NC2018_test","d":"The test set of a train/test pair from the toluene dataset from sGDML. To create the coupled cluster datasets, the data used for training the models were created by running ab initio MD in the NVT ensemble using the Nos\u00e9-Hoover thermostat at 500 K during a 200 ps simulation with a resolution of 0.5 fs. Energies and forces were recalculated using all-electron coupled cluster with single, double and perturbative triple excitations (CCSD(T)). The Dunning correlation-consistent basis set cc-pVDZ was used for malonaldehyde. All calculations were performed with the Psi4 software suite."}],["DS_o8ssxj2re4kx_0",{"n":"sGDML_Toluene_ccsdt_NC2018_train","d":"The train set of a train/test pair from the toluene dataset from sGDML. To create the coupled cluster datasets, the data used for training the models were created by running ab initio MD in the NVT ensemble using the Nos\u00e9-Hoover thermostat at 500 K during a 200 ps simulation with a resolution of 0.5 fs. Energies and forces were recalculated using all-electron coupled cluster with single, double and perturbative triple excitations (CCSD(T)). The Dunning correlation-consistent basis set cc-pVDZ was used for malonaldehyde. All calculations were performed with the Psi4 software suite."}],["DS_b4o9n4n8b7sp_0",{"n":"solute_strengthening_of_prism_edge_locations_in_Mg_alloys","d":"This dataset includes Mg and Mg-Zn alloy structures with solute atoms at the prism edge locations. The dataset was created to study the strengthening effect of solute atoms at the prism edge locations in Mg alloys."}],["DS_ctjgc03xdauc_0",{"n":"solvated_protein_fragments_JCTC_2019","d":"The solvated protein fragments dataset was generated as a partner benchmark dataset, along with SN2, for measuring the performance of machine learning models, in particular PhysNet, at describing chemical reactions, long-range interactions, and condensed phase systems. The dataset contains structures for all possible \"amons\" (hydrogen-saturated covalently bonded fragments) of up to eight heavy atoms (C, N, O, S) that can be derived from chemical graphs of proteins containing the 20 natural amino acids connected via peptide bonds or disulfide bridges. For amino acids that can occur in different charge states due to (de)protonation (i.e., carboxylic acids that can be negatively charged or amines that can be positively charged), all possible structures with up to a total charge of +-2e are included. In total, the dataset provides reference energies, forces, and dipole moments for 2,731,180 structures calculated at the revPBE-D3(BJ)/def2-TZVP level of theory using ORCA 4.0.1."}],["DS_qk4qfeka1e3m_0",{"n":"stable_and_metastable_phases_in_sputtered_CuInS2","d":"The chalcopyrite Cu(In,Ga)S2 has gained renewed interest in recent years due to its potential application in tandem solar cells. In this contribution, a combined theoretical and experimental approach is applied to investigate stable and metastable phases forming in sputtered CuInS2 (CIS) thin films. Ab initio calculations are performed to obtain formation energies, X-ray diffraction patterns, and Raman spectra of various CIS polytypes and related compounds. Multiple low-energy CIS structures with zinc-blende and wurtzite-derived lattices are identified and their XRD/Raman patterns are shown to contain many overlapping features, which could lead to misidentification unless the techniques are duly combined and analyzed. The results are verified against experimental XRD/Raman spectra measured on a series of CIS films with different compositions and treated at different temperatures, revealing the formation of several CIS polymorphs and secondary phases. The characteristic features and the mechanisms behind the formation of different phases are discussed with the focus on the thin-film photovoltaic application of CIS. The dataset contains structures and VASP output files used to derive the discussed trends. version 2"}],["DS_qde4v70hhmmj_0",{"n":"tmQM_wB97MV","d":"tmQM_wB97MV contains configurations from the tmQM dataset, with several structures from tmQM that were found to be missing hydrogens filtered out, and energies of all other structures recomputed at the wB97M-V/def2-SVPD level of DFT."}],["DS_4mjnowmrcqib_0",{"n":"vanadium_in_high_entropy_alloys_AM2020","d":"Dataset created for \"Vanadium is an optimal element for strengthening in both fcc and bcc high-entropy alloys\", to explore the effect of V in the high-entropy systems fcc Co-Cr-Fe-Mn-Ni-V and bcc Cr-Mo-Nb-Ta-V-W-Hf-Ti-Zr. Structures include pure V, misfit volumes of V in Ni, and misfit volumes of Ni2V random alloys"}],["DS_kl12pfupgv5e_0",{"n":"water_and_Cu+_synergy_in_selective_CO2_hydrogenation_to_methanol_over_Cu/MgO_catalysts","d":"This dataset was created to investigate the role of surface water and hydroxyl groups in facilitating spontaneous CO\u2082 activation at Cu\u207a sites and the formation of monodentate formate species in the context of using CO2 hydrogenation to produce methanol."}],["DS_p2aaxa2vfnr6_0",{"n":"water_ice_JCP_2020","d":"Starting from a single reference ab initio simulation, we use active learning to expand into new state points and to describe the quantum nature of the nuclei. The final model, trained on 814 reference calculations, yields excellent results under a range of conditions, from liquid water at ambient and elevated temperatures and pressures to different phases of ice, and the air-water interface \u2014 all including nuclear quantum effects."}],["DS_wm2xs5bhyzcx_0",{"n":"water_ice_NEP_2023","d":"The main part of the dataset consists of structures of liquid water at 300 K from first-principles molecular dynamics (FPMD) simulations using a hybrid density functional with dispersion corrections. The dataset is expanded to include nuclear quantum effects by adding structures from path-integral molecular dynamics (PIMD) simulations. The final dataset contains 814 structures of liquid water at different temperatures and pressures, water slab, and ice Ih and ice VIII. These systems cover a wide range of structural and dynamical properties of water and ice. This dataset builds on the dataset from Schran, et al (2020) https://doi.org/10.1063/5.0016004"}],["DS_mz0vl4zs7bex_0",{"n":"water_ice_PNAS_2021","d":"Dataset generated using a committee-based active learning strategy to build a training dataset for modeling complex aqueous systems."}],["DS_sytg0385f79a_0",{"n":"xxMD-CASSCF_test","d":"Test dataset from xxMD-CASSCF. The xxMD (Extended Excited-state Molecular Dynamics) dataset is a comprehensive collection of non-adiabatic trajectories encompassing several photo-sensitive molecules. This dataset challenges existing Neural Force Field (NFF) models with broader nuclear configuration spaces that span reactant, transition state, product, and conical intersection regions, making it more chemically representative than its contemporaries. xxMD is divided into two datasets, each with corresponding train, test and validation splits. xxMD-CASSCF contains calculations generated using state-averaged complete active state self-consistent field (SA-CASSCF) electronic theory. xxMD-DFT contains recalculated single-point spin-polarized (unrestricted) DFT values."}],["DS_qvljl3vtdp0f_0",{"n":"xxMD-CASSCF_train","d":"Training dataset from xxMD-CASSCF. The xxMD (Extended Excited-state Molecular Dynamics) dataset is a comprehensive collection of non-adiabatic trajectories encompassing several photo-sensitive molecules. This dataset challenges existing Neural Force Field (NFF) models with broader nuclear configuration spaces that span reactant, transition state, product, and conical intersection regions, making it more chemically representative than its contemporaries. xxMD is divided into two datasets, each with corresponding train, test and validation splits. xxMD-CASSCF contains calculations generated using state-averaged complete active state self-consistent field (SA-CASSCF) electronic theory. xxMD-DFT contains recalculated single-point spin-polarized (unrestricted) DFT values."}],["DS_e0nl35x6dl85_0",{"n":"xxMD-CASSCF_validation","d":"Validation dataset from xxMD-CASSCF. The xxMD (Extended Excited-state Molecular Dynamics) dataset is a comprehensive collection of non-adiabatic trajectories encompassing several photo-sensitive molecules. This dataset challenges existing Neural Force Field (NFF) models with broader nuclear configuration spaces that span reactant, transition state, product, and conical intersection regions, making it more chemically representative than its contemporaries. xxMD is divided into two datasets, each with corresponding train, test and validation splits. xxMD-CASSCF contains calculations generated using state-averaged complete active state self-consistent field (SA-CASSCF) electronic theory. xxMD-DFT contains recalculated single-point spin-polarized (unrestricted) DFT values."}],["DS_q007ninsdjvf_0",{"n":"xxMD-DFT_test","d":"Test dataset from xxMD-DFT. The xxMD (Extended Excited-state Molecular Dynamics) dataset is a comprehensive collection of non-adiabatic trajectories encompassing several photo-sensitive molecules. This dataset challenges existing Neural Force Field (NFF) models with broader nuclear configuration spaces that span reactant, transition state, product, and conical intersection regions, making it more chemically representative than its contemporaries. xxMD is divided into two datasets, each with corresponding train, test and validation splits. xxMD-CASSCF contains calculations generated using state-averaged complete active state self-consistent field (SA-CASSCF) electronic theory. xxMD-DFT contains recalculated single-point spin-polarized (unrestricted) DFT values."}],["DS_fblez5twbr07_0",{"n":"xxMD-DFT_train","d":"Training dataset from xxMD-DFT. The xxMD (Extended Excited-state Molecular Dynamics) dataset is a comprehensive collection of non-adiabatic trajectories encompassing several photo-sensitive molecules. This dataset challenges existing Neural Force Field (NFF) models with broader nuclear configuration spaces that span reactant, transition state, product, and conical intersection regions, making it more chemically representative than its contemporaries. xxMD is divided into two datasets, each with corresponding train, test and validation splits. xxMD-CASSCF contains calculations generated using state-averaged complete active state self-consistent field (SA-CASSCF) electronic theory. xxMD-DFT contains recalculated single-point spin-polarized (unrestricted) DFT values."}],["DS_d1nmrzy4csx1_0",{"n":"xxMD-DFT_validation","d":"Validation dataset from xxMD-DFT. The xxMD (Extended Excited-state Molecular Dynamics) dataset is a comprehensive collection of non-adiabatic trajectories encompassing several photo-sensitive molecules. This dataset challenges existing Neural Force Field (NFF) models with broader nuclear configuration spaces that span reactant, transition state, product, and conical intersection regions, making it more chemically representative than its contemporaries. xxMD is divided into two datasets, each with corresponding train, test and validation splits. xxMD-CASSCF contains calculations generated using state-averaged complete active state self-consistent field (SA-CASSCF) electronic theory. xxMD-DFT contains recalculated single-point spin-polarized (unrestricted) DFT values."}]];
